2019-06-12T19:37:34 INFO jox jox.py:233 Configure elastcisearch if enabled
2019-06-12T19:37:34 INFO jox jox.py:244 elasticsearch is now running
2019-06-12T19:37:34 INFO jox jox.py:247 Save JOX configuration to elasticsearch
2019-06-12T19:37:34 INFO jox jox.py:250 Delete the index of jox config from elasticsearch if exist
2019-06-12T19:37:34 INFO jox jox.py:253 Sending JOX configuration file to elasticsearch
2019-06-12T19:37:34 ERROR jox.es es.py:219 Error while getting the source of the index of jox_config from elasticsearch: NotFoundError(404, 'index_not_found_exception', 'no such index [jox_config]')
2019-06-12T19:37:34 INFO jox jox.py:257 The data of the index jox_config is addedd to elasticsearch
2019-06-12T19:37:34 INFO jox jox.py:266 Create resource Controller
2019-06-12T19:37:34 INFO jox jox.py:274 registering the following POP of type lxc as specified in jox_config: {'domain': '10.146.98.0/24', 'scope': 'private', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', 'ssh-password': '', 'zone': '1', 'ssh-user': '', 'managed-by': 'lxd', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty', '_Comment_3': 'ssh config', 'ssh-private-key': '', 'pop-name': 'default'}
2019-06-12T19:37:34 DEBUG jox jox.py:389 The following error raised while registering lxc pop with the config {'domain': '10.146.98.0/24', 'scope': 'private', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', 'ssh-password': '', 'zone': '1', 'ssh-user': '', 'managed-by': 'lxd', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty', '_Comment_3': 'ssh config', 'ssh-private-key': '', 'pop-name': 'default'}: 
 'lxc-anchor'
2019-06-12T19:37:34 INFO jox jox.py:274 registering the following POP of type phy as specified in jox_config: {'domain': '192.168.12.0/24', 'ssh-private-key': '', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty', 'ssh-password': '', 'zone': '1', 'ssh-user': '', 'managed-by': 'none', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', 'scope': 'private', '_Comment_3': 'ssh config', 'pop-name': 'zone-1-phy'}
2019-06-12T19:37:34 DEBUG jox jox.py:391 The phy pop with the following config is successfully regiestered: {'domain': '192.168.12.0/24', 'ssh-private-key': '', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty', 'ssh-password': '', 'zone': '1', 'ssh-user': '', 'managed-by': 'none', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', 'scope': 'private', '_Comment_3': 'ssh config', 'pop-name': 'zone-1-phy'}
2019-06-12T19:37:34 INFO jox jox.py:274 registering the following POP of type phy as specified in jox_config: {'domain': '172.24.0.0/16', 'ssh-private-key': '', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty', 'ssh-password': '', 'zone': '1', 'ssh-user': '', 'managed-by': 'none', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', 'scope': 'private', '_Comment_3': 'ssh config', 'pop-name': 'zone-2-phy'}
2019-06-12T19:37:34 DEBUG jox jox.py:391 The phy pop with the following config is successfully regiestered: {'domain': '172.24.0.0/16', 'ssh-private-key': '', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty', 'ssh-password': '', 'zone': '1', 'ssh-user': '', 'managed-by': 'none', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', 'scope': 'private', '_Comment_3': 'ssh config', 'pop-name': 'zone-2-phy'}
2019-06-12T19:37:34 INFO jox jox.py:274 registering the following POP of type phy as specified in jox_config: {'domain': '172.24.10.0/24', 'ssh-private-key': '', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty', 'ssh-password': '', 'zone': '1', 'ssh-user': '', 'managed-by': 'none', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', 'scope': 'private', '_Comment_3': 'ssh config', 'pop-name': 'zone-3-phy'}
2019-06-12T19:37:34 DEBUG jox jox.py:391 The phy pop with the following config is successfully regiestered: {'domain': '172.24.10.0/24', 'ssh-private-key': '', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty', 'ssh-password': '', 'zone': '1', 'ssh-user': '', 'managed-by': 'none', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', 'scope': 'private', '_Comment_3': 'ssh config', 'pop-name': 'zone-3-phy'}
2019-06-12T19:37:34 INFO jox jox.py:274 registering the following POP of type phy as specified in jox_config: {'domain': '172.24.11.0/24', 'ssh-private-key': '', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty', 'ssh-password': '', 'zone': '1', 'ssh-user': '', 'managed-by': 'none', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', 'scope': 'private', '_Comment_3': 'ssh config', 'pop-name': 'zone-4-phy'}
2019-06-12T19:37:34 DEBUG jox jox.py:391 The phy pop with the following config is successfully regiestered: {'domain': '172.24.11.0/24', 'ssh-private-key': '', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty', 'ssh-password': '', 'zone': '1', 'ssh-user': '', 'managed-by': 'none', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', 'scope': 'private', '_Comment_3': 'ssh config', 'pop-name': 'zone-4-phy'}
2019-06-12T19:37:34 INFO jox jox.py:274 registering the following POP of type kvm as specified in jox_config: {'domain': '192.168.122.0/24', 'ssh-private-key': '', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty', 'ssh-password': '', 'zone': '1', 'ssh-user': '', 'managed-by': 'kvm', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', 'scope': 'private', '_Comment_3': 'ssh config', 'pop-name': 'default'}
2019-06-12T19:37:34 DEBUG jox jox.py:391 The kvm pop with the following config is successfully regiestered: {'domain': '192.168.122.0/24', 'ssh-private-key': '', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty', 'ssh-password': '', 'zone': '1', 'ssh-user': '', 'managed-by': 'kvm', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', 'scope': 'private', '_Comment_3': 'ssh config', 'pop-name': 'default'}
2019-06-12T19:37:34 INFO jox jox.py:281 Clouds Controller is successfully loaded!
2019-06-12T19:37:34 DEBUG jox.RO.rsourceController resource_controller.py:539 jcloud localhost was added to jox
2019-06-12T19:37:34 DEBUG jox jox.py:405 The following cloud is successfully  added
2019-06-12T19:37:34 DEBUG jox.RO.rsourceController resource_controller.py:539 jcloud nymphe-edu was added to jox
2019-06-12T19:37:34 DEBUG jox jox.py:405 The following cloud is successfully  added
2019-06-12T19:37:34 INFO jox jox.py:293 All clouds were successfully added!
2019-06-12T19:37:34 INFO jox jox.py:297 Creating slice controller
2019-06-12T19:37:34 INFO jox.NetworkSliceController nsi_controller.py:41 Initial configuration of network slice controller
2019-06-12T19:37:34 INFO jox.NetworkSliceController nsi_controller.py:62 The resource controller is built
2019-06-12T19:37:34 INFO jox jox.py:301 Creating sub-slice controller
2019-06-12T19:37:34 INFO jox jox.py:309 Slices Controller was successfully loaded!
2019-06-12T19:37:34 INFO jox jox.py:314 JOX loading time: 0.3174874782562256 
2019-06-12T19:37:34 INFO jox jox.py:324 JOX Web API loaded!
2019-06-12T19:38:19 INFO jox jox.py:410 Adding the slice mosaic5g-oai-4g.yaml
2019-06-12T19:38:19 INFO jox.NetworkSliceController nsi_controller.py:102 Adding the slice mosaic5g-oai-4g.yaml
2019-06-12T19:38:19 ERROR jox.es es.py:219 Error while getting the source of the index of slice_keys_oai-4g from elasticsearch: NotFoundError(404, 'index_not_found_exception', 'no such index [slice_keys_oai-4g]')
2019-06-12T19:38:20 ERROR jox.es es.py:219 Error while getting the source of the index of slice_monitor_oai-4g from elasticsearch: NotFoundError(404, 'index_not_found_exception', 'no such index [slice_monitor_oai-4g]')
2019-06-12T19:38:20 INFO jox.templateManager template_manager.py:128 Deleting the index oai-4g from elasticsearch if alredy exist
2019-06-12T19:38:20 DEBUG jox.templateManager template_manager.py:129 Deleting the index oai-4g from elasticsearch if alredy exist
2019-06-12T19:38:20 INFO jox.templateManager template_manager.py:132 Saving the index oai-4g to elasticsearch
2019-06-12T19:38:20 DEBUG jox.templateManager template_manager.py:133 Saving the index oai-4g to elasticsearch
2019-06-12T19:38:20 ERROR jox.es es.py:219 Error while getting the source of the index of oai-4g from elasticsearch: NotFoundError(404, 'index_not_found_exception', 'no such index [oai-4g]')
2019-06-12T19:38:20 INFO jox.templateManager template_manager.py:155 Deleting the index nssi_1 from elasticsearch if already exist
2019-06-12T19:38:20 DEBUG jox.templateManager template_manager.py:156 Deleting the index nssi_1 from elasticsearch if already exist
2019-06-12T19:38:20 INFO jox.templateManager template_manager.py:161 Saving the index nssi_1 to elasticsearch
2019-06-12T19:38:20 DEBUG jox.templateManager template_manager.py:162 Saving the index nssi_1 to elasticsearch
2019-06-12T19:38:20 ERROR jox.es es.py:219 Error while getting the source of the index of nssi_1 from elasticsearch: NotFoundError(404, 'index_not_found_exception', 'no such index [nssi_1]')
2019-06-12T19:38:20 INFO jox.templateManager template_manager.py:155 Deleting the index nssi_2 from elasticsearch if already exist
2019-06-12T19:38:20 DEBUG jox.templateManager template_manager.py:156 Deleting the index nssi_2 from elasticsearch if already exist
2019-06-12T19:38:21 INFO jox.templateManager template_manager.py:161 Saving the index nssi_2 to elasticsearch
2019-06-12T19:38:21 DEBUG jox.templateManager template_manager.py:162 Saving the index nssi_2 to elasticsearch
2019-06-12T19:38:21 ERROR jox.es es.py:219 Error while getting the source of the index of nssi_2 from elasticsearch: NotFoundError(404, 'index_not_found_exception', 'no such index [nssi_2]')
2019-06-12T19:38:21 INFO jox.templateManager template_manager.py:155 Deleting the index nssi_3 from elasticsearch if already exist
2019-06-12T19:38:21 DEBUG jox.templateManager template_manager.py:156 Deleting the index nssi_3 from elasticsearch if already exist
2019-06-12T19:38:21 INFO jox.templateManager template_manager.py:161 Saving the index nssi_3 to elasticsearch
2019-06-12T19:38:21 DEBUG jox.templateManager template_manager.py:162 Saving the index nssi_3 to elasticsearch
2019-06-12T19:38:21 ERROR jox.es es.py:219 Error while getting the source of the index of nssi_3 from elasticsearch: NotFoundError(404, 'index_not_found_exception', 'no such index [nssi_3]')
2019-06-12T19:38:21 ERROR jox.es es.py:219 Error while getting the source of the index of subslice_monitor_nssi_1 from elasticsearch: NotFoundError(404, 'index_not_found_exception', 'no such index [subslice_monitor_nssi_1]')
2019-06-12T19:38:22 INFO jox.jslice slice.py:105 trying to add the subslice nssi_1
2019-06-12T19:38:22 INFO jox.subslice subslice.py:118 add the models on which the services of the  NSSI nssi_1 will be deployed
2019-06-12T19:38:22 INFO jox.jmodel model.py:143 The juju model nymphe-edu:default is configured
2019-06-12T19:38:22 DEBUG jox.jmodel model.py:144 The ID of the juju model nymphe-edu:default is f5240599-4e6f-44ff-8951-2419243bf1f6
2019-06-12T19:38:22 INFO jox.JCloud cloud.py:170 Connecting to controller
2019-06-12T19:38:22 INFO jox.JCloud cloud.py:170 Connecting to controller
2019-06-12T19:38:22 INFO jox.JCloud cloud.py:170 Connecting to controller
2019-06-12T19:38:22 INFO jox.RO.rsourceController resource_controller.py:342 Adding machine for the application flexran
2019-06-12T19:38:22 INFO jox.JCloud cloud.py:170 Connecting to controller
2019-06-12T19:38:22 INFO jox.RO.rsourceController resource_controller.py:441 Adding kvm machine whose id is: machine_name= 1:kvm, mid_ro = VDU_flexran
2019-06-12T19:38:23 DEBUG jox.KvmDriver.default vimdriver.py:723 0 time trying to connect to juju model:nymphe-edu:admin/default to add lxc machine
2019-06-12T19:38:23 DEBUG jox.KvmDriver.default vimdriver.py:726 Successful connection to the model nymphe-edu:admin/default 
2019-06-12T19:38:24 DEBUG jox.KvmDriver.default vimdriver.py:832 The machine VDU_flexran with juju id 49 is already added to juju model
2019-06-12T19:38:24 DEBUG jox.RO.rsourceController resource_controller.py:453 ('flexran', ' kvm machine added')
2019-06-12T19:38:24 INFO jox.RO.rsourceController resource_controller.py:342 Adding machine for the application oai-hss
2019-06-12T19:38:24 INFO jox.RO.rsourceController resource_controller.py:441 Adding kvm machine whose id is: machine_name= 2:kvm, mid_ro = VDU_oai_hss
2019-06-12T19:38:24 DEBUG jox.KvmDriver.default vimdriver.py:723 0 time trying to connect to juju model:nymphe-edu:admin/default to add lxc machine
2019-06-12T19:38:24 DEBUG jox.KvmDriver.default vimdriver.py:726 Successful connection to the model nymphe-edu:admin/default 
2019-06-12T19:38:24 DEBUG jox.KvmDriver.default vimdriver.py:832 The machine VDU_oai_hss with juju id 51 is already added to juju model
2019-06-12T19:38:24 DEBUG jox.RO.rsourceController resource_controller.py:453 ('oai-hss', ' kvm machine added')
2019-06-12T19:38:24 INFO jox.RO.rsourceController resource_controller.py:342 Adding machine for the application mysql
2019-06-12T19:38:24 INFO jox.RO.rsourceController resource_controller.py:441 Adding kvm machine whose id is: machine_name= 3:kvm, mid_ro = VDU_mysql
2019-06-12T19:38:24 DEBUG jox.KvmDriver.default vimdriver.py:723 0 time trying to connect to juju model:nymphe-edu:admin/default to add lxc machine
2019-06-12T19:38:24 DEBUG jox.KvmDriver.default vimdriver.py:726 Successful connection to the model nymphe-edu:admin/default 
2019-06-12T19:38:25 DEBUG jox.KvmDriver.default vimdriver.py:832 The machine VDU_mysql with juju id 48 is already added to juju model
2019-06-12T19:38:25 DEBUG jox.RO.rsourceController resource_controller.py:453 ('mysql', ' kvm machine added')
2019-06-12T19:38:25 DEBUG jox.RO.rsourceController resource_controller.py:504 Waiting for the machines to start to deploy the services
2019-06-12T19:38:25 ERROR jox.nssiController nssi_controller.py:132 module 'src.common.config.gv' has no attribute 'PHY_2'
2019-06-12T19:38:25 ERROR jox.nssiController nssi_controller.py:133 Traceback (most recent call last):
  File "/home/psoque/JoX/jox-plugin/src/core/nso/nssi/nssi_controller.py", line 127, in add_subslice
    new_slice.build_jsubslice(subslice_config, self.resourceController, list_all_jujuModel_attachedWatcher)
  File "/home/psoque/JoX/jox-plugin/src/core/nso/nssi/subslice.py", line 121, in build_jsubslice
    raise ex
  File "/home/psoque/JoX/jox-plugin/src/core/nso/nssi/subslice.py", line 119, in build_jsubslice
    self.add_models(subslice_config, resourceController, list_all_jujuModel_attachedWatcher)
  File "/home/psoque/JoX/jox-plugin/src/core/nso/nssi/subslice.py", line 241, in add_models
    raise ex
  File "/home/psoque/JoX/jox-plugin/src/core/nso/nssi/subslice.py", line 234, in add_models
    self.add_model(subslice_config, resourceController, juju_cloud, juju_model, list_all_jujuModel_attachedWatcher)
  File "/home/psoque/JoX/jox-plugin/src/core/nso/nssi/subslice.py", line 265, in add_model
    raise ex
  File "/home/psoque/JoX/jox-plugin/src/core/nso/nssi/subslice.py", line 262, in add_model
    resourceController, juju_cloud, juju_model, Watcher_attached, self.subslice_machines)
  File "/home/psoque/JoX/jox-plugin/src/core/nso/nssi/model.py", line 166, in build_and_deploy
    raise ex
  File "/home/psoque/JoX/jox-plugin/src/core/nso/nssi/model.py", line 150, in build_and_deploy
    list_machine = resourceController.add_machines(self.machines_config, self.services_config, self.subslice_name, self.model_name, self.cloud_name, self.slice_name)
  File "/home/psoque/JoX/jox-plugin/src/core/ro/resource_controller.py", line 533, in add_machines
    raise ex
  File "/home/psoque/JoX/jox-plugin/src/core/ro/resource_controller.py", line 523, in add_machines
    if [(self.gv.PHY == pop_type or self.gv.PHY_2 == pop_type) for pop_type in machine_types]:
  File "/home/psoque/JoX/jox-plugin/src/core/ro/resource_controller.py", line 523, in <listcomp>
    if [(self.gv.PHY == pop_type or self.gv.PHY_2 == pop_type) for pop_type in machine_types]:
AttributeError: module 'src.common.config.gv' has no attribute 'PHY_2'

2019-06-12T19:38:25 INFO jox.jslice slice.py:111 the subslice nssi_1 is NOT added. Exit...
2019-06-12T19:38:25 ERROR jox.es es.py:219 Error while getting the source of the index of subslice_monitor_nssi_2 from elasticsearch: NotFoundError(404, 'index_not_found_exception', 'no such index [subslice_monitor_nssi_2]')
2019-06-12T19:38:25 INFO jox.jslice slice.py:105 trying to add the subslice nssi_2
2019-06-12T19:38:25 INFO jox.subslice subslice.py:118 add the models on which the services of the  NSSI nssi_2 will be deployed
2019-06-12T19:38:26 INFO jox.jmodel model.py:143 The juju model nymphe-edu:default is configured
2019-06-12T19:38:26 DEBUG jox.jmodel model.py:144 The ID of the juju model nymphe-edu:default is f5240599-4e6f-44ff-8951-2419243bf1f6
2019-06-12T19:38:26 INFO jox.RO.rsourceController resource_controller.py:342 Adding machine for the application oai-mme
2019-06-12T19:38:26 INFO jox.RO.rsourceController resource_controller.py:441 Adding kvm machine whose id is: machine_name= 4:kvm, mid_ro = VDU_oai-mme
2019-06-12T19:38:26 DEBUG jox.KvmDriver.default vimdriver.py:723 0 time trying to connect to juju model:nymphe-edu:admin/default to add lxc machine
2019-06-12T19:38:26 DEBUG jox.KvmDriver.default vimdriver.py:726 Successful connection to the model nymphe-edu:admin/default 
2019-06-12T19:38:26 DEBUG jox.KvmDriver.default vimdriver.py:832 The machine VDU_oai-mme with juju id 47 is already added to juju model
2019-06-12T19:38:26 DEBUG jox.RO.rsourceController resource_controller.py:453 ('oai-mme', ' kvm machine added')
2019-06-12T19:38:26 INFO jox.RO.rsourceController resource_controller.py:342 Adding machine for the application oai-spgw
2019-06-12T19:38:26 INFO jox.RO.rsourceController resource_controller.py:441 Adding kvm machine whose id is: machine_name= 5:kvm, mid_ro = VDU_oai-spgw
2019-06-12T19:38:26 DEBUG jox.KvmDriver.default vimdriver.py:723 0 time trying to connect to juju model:nymphe-edu:admin/default to add lxc machine
2019-06-12T19:38:26 DEBUG jox.KvmDriver.default vimdriver.py:726 Successful connection to the model nymphe-edu:admin/default 
2019-06-12T19:38:27 DEBUG jox.KvmDriver.default vimdriver.py:832 The machine VDU_oai-spgw with juju id 50 is already added to juju model
2019-06-12T19:38:27 DEBUG jox.RO.rsourceController resource_controller.py:453 ('oai-spgw', ' kvm machine added')
2019-06-12T19:38:27 DEBUG jox.RO.rsourceController resource_controller.py:504 Waiting for the machines to start to deploy the services
2019-06-12T19:38:27 ERROR jox.nssiController nssi_controller.py:132 module 'src.common.config.gv' has no attribute 'PHY_2'
2019-06-12T19:38:27 ERROR jox.nssiController nssi_controller.py:133 Traceback (most recent call last):
  File "/home/psoque/JoX/jox-plugin/src/core/nso/nssi/nssi_controller.py", line 127, in add_subslice
    new_slice.build_jsubslice(subslice_config, self.resourceController, list_all_jujuModel_attachedWatcher)
  File "/home/psoque/JoX/jox-plugin/src/core/nso/nssi/subslice.py", line 121, in build_jsubslice
    raise ex
  File "/home/psoque/JoX/jox-plugin/src/core/nso/nssi/subslice.py", line 119, in build_jsubslice
    self.add_models(subslice_config, resourceController, list_all_jujuModel_attachedWatcher)
  File "/home/psoque/JoX/jox-plugin/src/core/nso/nssi/subslice.py", line 241, in add_models
    raise ex
  File "/home/psoque/JoX/jox-plugin/src/core/nso/nssi/subslice.py", line 234, in add_models
    self.add_model(subslice_config, resourceController, juju_cloud, juju_model, list_all_jujuModel_attachedWatcher)
  File "/home/psoque/JoX/jox-plugin/src/core/nso/nssi/subslice.py", line 265, in add_model
    raise ex
  File "/home/psoque/JoX/jox-plugin/src/core/nso/nssi/subslice.py", line 262, in add_model
    resourceController, juju_cloud, juju_model, Watcher_attached, self.subslice_machines)
  File "/home/psoque/JoX/jox-plugin/src/core/nso/nssi/model.py", line 166, in build_and_deploy
    raise ex
  File "/home/psoque/JoX/jox-plugin/src/core/nso/nssi/model.py", line 150, in build_and_deploy
    list_machine = resourceController.add_machines(self.machines_config, self.services_config, self.subslice_name, self.model_name, self.cloud_name, self.slice_name)
  File "/home/psoque/JoX/jox-plugin/src/core/ro/resource_controller.py", line 533, in add_machines
    raise ex
  File "/home/psoque/JoX/jox-plugin/src/core/ro/resource_controller.py", line 523, in add_machines
    if [(self.gv.PHY == pop_type or self.gv.PHY_2 == pop_type) for pop_type in machine_types]:
  File "/home/psoque/JoX/jox-plugin/src/core/ro/resource_controller.py", line 523, in <listcomp>
    if [(self.gv.PHY == pop_type or self.gv.PHY_2 == pop_type) for pop_type in machine_types]:
AttributeError: module 'src.common.config.gv' has no attribute 'PHY_2'

2019-06-12T19:38:27 INFO jox.jslice slice.py:111 the subslice nssi_2 is NOT added. Exit...
2019-06-12T19:38:27 ERROR jox.es es.py:219 Error while getting the source of the index of subslice_monitor_nssi_3 from elasticsearch: NotFoundError(404, 'index_not_found_exception', 'no such index [subslice_monitor_nssi_3]')
2019-06-12T19:38:27 INFO jox.jslice slice.py:105 trying to add the subslice nssi_3
2019-06-12T19:38:27 INFO jox.subslice subslice.py:118 add the models on which the services of the  NSSI nssi_3 will be deployed
2019-06-12T19:38:28 INFO jox.jmodel model.py:143 The juju model nymphe-edu:default is configured
2019-06-12T19:38:28 DEBUG jox.jmodel model.py:144 The ID of the juju model nymphe-edu:default is f5240599-4e6f-44ff-8951-2419243bf1f6
2019-06-12T19:38:28 INFO jox.RO.rsourceController resource_controller.py:342 Adding machine for the application oai-ran
2019-06-12T19:38:28 INFO jox.RO.rsourceController resource_controller.py:486 Adding physical machine whose id is: machine_name= 6:phy, mid_ro = VDU_oai-ran
2019-06-12T19:38:28 DEBUG jox.PhyDriver.zone-1-phy vimdriver.py:1079 0 time trying to connect to juju model:nymphe-edu:admin/default to add lxc machine
2019-06-12T19:38:28 DEBUG jox.PhyDriver.zone-1-phy vimdriver.py:1082 Successful connection to the model nymphe-edu:admin/default 
2019-06-12T19:38:28 DEBUG jox.RO.rsourceController resource_controller.py:498 ('oai-ran', ' kvm machine added')
2019-06-12T19:38:28 DEBUG jox.RO.rsourceController resource_controller.py:504 Waiting for the machines to start to deploy the services
2019-06-12T19:38:28 INFO jox.jmodel model.py:202 Deploying the application oai-ran
2019-06-12 19:38:37,675 - jox.JujuModelServiceController - INFO - The servce oai-ran is deployed
2019-06-12 19:38:37,675 - jox.JujuModelServiceController - INFO - The servce oai-ran is deployed
2019-06-12 19:38:37,675 - jox.JujuModelServiceController - INFO - The servce oai-ran is deployed
2019-06-12T19:38:37 INFO jox.JujuModelServiceController juju2_plugin.py:285 The servce oai-ran is deployed
2019-06-12 19:38:37,678 - jox.JujuModelServiceController - DEBUG - The servce oai-ran is deployed
2019-06-12 19:38:37,678 - jox.JujuModelServiceController - DEBUG - The servce oai-ran is deployed
2019-06-12 19:38:37,678 - jox.JujuModelServiceController - DEBUG - The servce oai-ran is deployed
2019-06-12T19:38:37 DEBUG jox.JujuModelServiceController juju2_plugin.py:286 The servce oai-ran is deployed
2019-06-12T19:38:37 INFO jox.nssiController nssi_controller.py:129 Slice nssi_3 was added
2019-06-12T19:38:37 INFO jox.jslice slice.py:108 the subslice nssi_3 is successfully added
2019-06-12T19:38:37 INFO jox.jslice slice.py:115 adding entre-nssi relations
2019-06-12T19:38:37 INFO jox.jslice slice.py:81 getting the inter relations of the slice oai-4g
2019-06-12T19:38:37 DEBUG jox.jslice slice.py:156 adding relation between oai-ran from the nssi nssi_3 and flexran from the nssi nssi_1
2019-06-12T19:38:37 INFO jox jox.py:233 Configure elastcisearch if enabled
2019-06-12T19:38:37 INFO jox jox.py:244 elasticsearch is now running
2019-06-12T19:38:37 INFO jox jox.py:247 Save JOX configuration to elasticsearch
2019-06-12T19:38:37 INFO jox jox.py:250 Delete the index of jox config from elasticsearch if exist
2019-06-12T19:38:37 INFO jox jox.py:253 Sending JOX configuration file to elasticsearch
2019-06-12T19:38:37 ERROR jox.es es.py:219 Error while getting the source of the index of jox_config from elasticsearch: NotFoundError(404, 'index_not_found_exception', 'no such index [jox_config]')
2019-06-12T19:38:38 INFO jox jox.py:257 The data of the index jox_config is addedd to elasticsearch
2019-06-12T19:38:38 INFO jox jox.py:266 Create resource Controller
2019-06-12T19:38:38 INFO jox jox.py:274 registering the following POP of type phy as specified in jox_config: {'zone': '1', '_Comment_3': 'ssh config', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', 'ssh-user': '', 'ssh-password': '', 'managed-by': 'none', 'pop-name': 'zone-1-phy', 'scope': 'private', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty', 'ssh-private-key': '', 'domain': '192.168.12.0/24'}
2019-06-12T19:38:38 DEBUG jox jox.py:391 The phy pop with the following config is successfully regiestered: {'zone': '1', '_Comment_3': 'ssh config', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', 'ssh-user': '', 'ssh-password': '', 'managed-by': 'none', 'pop-name': 'zone-1-phy', 'scope': 'private', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty', 'ssh-private-key': '', 'domain': '192.168.12.0/24'}
2019-06-12T19:38:38 INFO jox jox.py:274 registering the following POP of type phy as specified in jox_config: {'zone': '1', '_Comment_3': 'ssh config', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', 'ssh-user': '', 'ssh-password': '', 'managed-by': 'none', 'pop-name': 'zone-2-phy', 'scope': 'private', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty', 'ssh-private-key': '', 'domain': '172.24.0.0/16'}
2019-06-12T19:38:38 DEBUG jox jox.py:391 The phy pop with the following config is successfully regiestered: {'zone': '1', '_Comment_3': 'ssh config', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', 'ssh-user': '', 'ssh-password': '', 'managed-by': 'none', 'pop-name': 'zone-2-phy', 'scope': 'private', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty', 'ssh-private-key': '', 'domain': '172.24.0.0/16'}
2019-06-12T19:38:38 INFO jox jox.py:274 registering the following POP of type phy as specified in jox_config: {'zone': '1', '_Comment_3': 'ssh config', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', 'ssh-user': '', 'ssh-password': '', 'managed-by': 'none', 'pop-name': 'zone-3-phy', 'scope': 'private', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty', 'ssh-private-key': '', 'domain': '172.24.10.0/24'}
2019-06-12T19:38:38 DEBUG jox jox.py:391 The phy pop with the following config is successfully regiestered: {'zone': '1', '_Comment_3': 'ssh config', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', 'ssh-user': '', 'ssh-password': '', 'managed-by': 'none', 'pop-name': 'zone-3-phy', 'scope': 'private', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty', 'ssh-private-key': '', 'domain': '172.24.10.0/24'}
2019-06-12T19:38:38 INFO jox jox.py:274 registering the following POP of type phy as specified in jox_config: {'zone': '1', '_Comment_3': 'ssh config', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', 'ssh-user': '', 'ssh-password': '', 'managed-by': 'none', 'pop-name': 'zone-4-phy', 'scope': 'private', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty', 'ssh-private-key': '', 'domain': '172.24.11.0/24'}
2019-06-12T19:38:38 DEBUG jox jox.py:391 The phy pop with the following config is successfully regiestered: {'zone': '1', '_Comment_3': 'ssh config', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', 'ssh-user': '', 'ssh-password': '', 'managed-by': 'none', 'pop-name': 'zone-4-phy', 'scope': 'private', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty', 'ssh-private-key': '', 'domain': '172.24.11.0/24'}
2019-06-12T19:38:38 INFO jox jox.py:274 registering the following POP of type kvm as specified in jox_config: {'zone': '1', '_Comment_3': 'ssh config', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', 'ssh-user': '', 'ssh-password': '', 'managed-by': 'kvm', 'pop-name': 'default', 'scope': 'private', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty', 'ssh-private-key': '', 'domain': '192.168.122.0/24'}
2019-06-12T19:38:38 DEBUG jox jox.py:391 The kvm pop with the following config is successfully regiestered: {'zone': '1', '_Comment_3': 'ssh config', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', 'ssh-user': '', 'ssh-password': '', 'managed-by': 'kvm', 'pop-name': 'default', 'scope': 'private', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty', 'ssh-private-key': '', 'domain': '192.168.122.0/24'}
2019-06-12T19:38:38 INFO jox jox.py:274 registering the following POP of type lxc as specified in jox_config: {'zone': '1', '_Comment_3': 'ssh config', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', 'ssh-user': '', 'ssh-password': '', 'managed-by': 'lxd', 'pop-name': 'default', 'scope': 'private', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty', 'ssh-private-key': '', 'domain': '10.146.98.0/24'}
2019-06-12T19:38:38 DEBUG jox jox.py:389 The following error raised while registering lxc pop with the config {'zone': '1', '_Comment_3': 'ssh config', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', 'ssh-user': '', 'ssh-password': '', 'managed-by': 'lxd', 'pop-name': 'default', 'scope': 'private', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty', 'ssh-private-key': '', 'domain': '10.146.98.0/24'}: 
 'lxc-anchor'
2019-06-12T19:38:38 INFO jox jox.py:281 Clouds Controller is successfully loaded!
2019-06-12T19:38:38 DEBUG jox.RO.rsourceController resource_controller.py:539 jcloud localhost was added to jox
2019-06-12T19:38:38 DEBUG jox jox.py:405 The following cloud is successfully  added
2019-06-12T19:38:38 DEBUG jox.RO.rsourceController resource_controller.py:539 jcloud nymphe-edu was added to jox
2019-06-12T19:38:38 DEBUG jox jox.py:405 The following cloud is successfully  added
2019-06-12T19:38:38 INFO jox jox.py:293 All clouds were successfully added!
2019-06-12T19:38:38 INFO jox jox.py:297 Creating slice controller
2019-06-12T19:38:38 INFO jox.NetworkSliceController nsi_controller.py:41 Initial configuration of network slice controller
2019-06-12T19:38:38 INFO jox.NetworkSliceController nsi_controller.py:62 The resource controller is built
2019-06-12T19:38:38 INFO jox jox.py:301 Creating sub-slice controller
2019-06-12T19:38:38 INFO jox jox.py:309 Slices Controller was successfully loaded!
2019-06-12T19:38:38 INFO jox jox.py:314 JOX loading time: 0.3145451545715332 
2019-06-12T19:38:38 INFO jox jox.py:324 JOX Web API loaded!
2019-06-12T19:38:47 DEBUG jox.jslice slice.py:156 adding relation between oai-hss from the nssi nssi_1 and oai-mme from the nssi nssi_2
2019-06-12T19:38:58 DEBUG jox.jslice slice.py:156 adding relation between oai-mme from the nssi nssi_2 and oai-ran from the nssi nssi_3
2019-06-12T19:39:08 INFO jox.NetworkSliceController nsi_controller.py:114 The slice mosaic5g-oai-4g.yaml is not deployed, due to the failure of adding at least one subslice
2019-06-12T19:39:08 DEBUG jox.NetworkSliceController nsi_controller.py:115 The slice mosaic5g-oai-4g.yaml is not deployed, due to the failure of adding at least one subslice
2019-06-12T19:39:08 ERROR jox.NetworkSliceController nsi_controller.py:116 The slice mosaic5g-oai-4g.yaml is not deployed, due to the failure of adding at least one subslice
2019-06-12T19:39:12 INFO jox jox.py:233 Configure elastcisearch if enabled
2019-06-12T19:39:12 INFO jox jox.py:244 elasticsearch is now running
2019-06-12T19:39:12 INFO jox jox.py:247 Save JOX configuration to elasticsearch
2019-06-12T19:39:12 INFO jox jox.py:250 Delete the index of jox config from elasticsearch if exist
2019-06-12T19:39:12 INFO jox jox.py:253 Sending JOX configuration file to elasticsearch
2019-06-12T19:39:12 ERROR jox.es es.py:219 Error while getting the source of the index of jox_config from elasticsearch: NotFoundError(404, 'index_not_found_exception', 'no such index [jox_config]')
2019-06-12T19:39:12 INFO jox jox.py:257 The data of the index jox_config is addedd to elasticsearch
2019-06-12T19:39:12 INFO jox jox.py:266 Create resource Controller
2019-06-12T19:39:12 INFO jox jox.py:274 registering the following POP of type kvm as specified in jox_config: {'scope': 'private', 'ssh-password': '', 'ssh-private-key': '', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty', 'domain': '192.168.122.0/24', 'pop-name': 'default', 'zone': '1', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', '_Comment_3': 'ssh config', 'ssh-user': '', 'managed-by': 'kvm'}
2019-06-12T19:39:12 DEBUG jox jox.py:391 The kvm pop with the following config is successfully regiestered: {'scope': 'private', 'ssh-password': '', 'ssh-private-key': '', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty', 'domain': '192.168.122.0/24', 'pop-name': 'default', 'zone': '1', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', '_Comment_3': 'ssh config', 'ssh-user': '', 'managed-by': 'kvm'}
2019-06-12T19:39:12 INFO jox jox.py:274 registering the following POP of type lxc as specified in jox_config: {'scope': 'private', 'ssh-password': '', 'ssh-private-key': '', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty', 'domain': '10.146.98.0/24', 'pop-name': 'default', 'zone': '1', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', '_Comment_3': 'ssh config', 'ssh-user': '', 'managed-by': 'lxd'}
2019-06-12T19:39:12 DEBUG jox jox.py:389 The following error raised while registering lxc pop with the config {'scope': 'private', 'ssh-password': '', 'ssh-private-key': '', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty', 'domain': '10.146.98.0/24', 'pop-name': 'default', 'zone': '1', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', '_Comment_3': 'ssh config', 'ssh-user': '', 'managed-by': 'lxd'}: 
 'lxc-anchor'
2019-06-12T19:39:12 INFO jox jox.py:274 registering the following POP of type phy as specified in jox_config: {'scope': 'private', 'ssh-password': '', 'ssh-private-key': '', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty', 'domain': '192.168.12.0/24', 'pop-name': 'zone-1-phy', 'zone': '1', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', '_Comment_3': 'ssh config', 'ssh-user': '', 'managed-by': 'none'}
2019-06-12T19:39:12 DEBUG jox jox.py:391 The phy pop with the following config is successfully regiestered: {'scope': 'private', 'ssh-password': '', 'ssh-private-key': '', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty', 'domain': '192.168.12.0/24', 'pop-name': 'zone-1-phy', 'zone': '1', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', '_Comment_3': 'ssh config', 'ssh-user': '', 'managed-by': 'none'}
2019-06-12T19:39:12 INFO jox jox.py:274 registering the following POP of type phy as specified in jox_config: {'scope': 'private', 'ssh-password': '', 'ssh-private-key': '', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty', 'domain': '172.24.0.0/16', 'pop-name': 'zone-2-phy', 'zone': '1', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', '_Comment_3': 'ssh config', 'ssh-user': '', 'managed-by': 'none'}
2019-06-12T19:39:12 DEBUG jox jox.py:391 The phy pop with the following config is successfully regiestered: {'scope': 'private', 'ssh-password': '', 'ssh-private-key': '', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty', 'domain': '172.24.0.0/16', 'pop-name': 'zone-2-phy', 'zone': '1', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', '_Comment_3': 'ssh config', 'ssh-user': '', 'managed-by': 'none'}
2019-06-12T19:39:12 INFO jox jox.py:274 registering the following POP of type phy as specified in jox_config: {'scope': 'private', 'ssh-password': '', 'ssh-private-key': '', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty', 'domain': '172.24.10.0/24', 'pop-name': 'zone-3-phy', 'zone': '1', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', '_Comment_3': 'ssh config', 'ssh-user': '', 'managed-by': 'none'}
2019-06-12T19:39:12 DEBUG jox jox.py:391 The phy pop with the following config is successfully regiestered: {'scope': 'private', 'ssh-password': '', 'ssh-private-key': '', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty', 'domain': '172.24.10.0/24', 'pop-name': 'zone-3-phy', 'zone': '1', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', '_Comment_3': 'ssh config', 'ssh-user': '', 'managed-by': 'none'}
2019-06-12T19:39:12 INFO jox jox.py:274 registering the following POP of type phy as specified in jox_config: {'scope': 'private', 'ssh-password': '', 'ssh-private-key': '', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty', 'domain': '172.24.11.0/24', 'pop-name': 'zone-4-phy', 'zone': '1', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', '_Comment_3': 'ssh config', 'ssh-user': '', 'managed-by': 'none'}
2019-06-12T19:39:12 DEBUG jox jox.py:391 The phy pop with the following config is successfully regiestered: {'scope': 'private', 'ssh-password': '', 'ssh-private-key': '', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty', 'domain': '172.24.11.0/24', 'pop-name': 'zone-4-phy', 'zone': '1', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', '_Comment_3': 'ssh config', 'ssh-user': '', 'managed-by': 'none'}
2019-06-12T19:39:12 INFO jox jox.py:281 Clouds Controller is successfully loaded!
2019-06-12T19:39:12 DEBUG jox.RO.rsourceController resource_controller.py:539 jcloud localhost was added to jox
2019-06-12T19:39:12 DEBUG jox jox.py:405 The following cloud is successfully  added
2019-06-12T19:39:12 DEBUG jox.RO.rsourceController resource_controller.py:539 jcloud nymphe-edu was added to jox
2019-06-12T19:39:12 DEBUG jox jox.py:405 The following cloud is successfully  added
2019-06-12T19:39:12 INFO jox jox.py:293 All clouds were successfully added!
2019-06-12T19:39:12 INFO jox jox.py:297 Creating slice controller
2019-06-12T19:39:12 INFO jox.NetworkSliceController nsi_controller.py:41 Initial configuration of network slice controller
2019-06-12T19:39:12 INFO jox.NetworkSliceController nsi_controller.py:62 The resource controller is built
2019-06-12T19:39:12 INFO jox jox.py:301 Creating sub-slice controller
2019-06-12T19:39:12 INFO jox jox.py:309 Slices Controller was successfully loaded!
2019-06-12T19:39:12 INFO jox jox.py:314 JOX loading time: 0.31070423126220703 
2019-06-12T19:39:12 INFO jox jox.py:324 JOX Web API loaded!
2019-06-12T19:39:35 INFO jox jox.py:410 Adding the slice mosaic5g-oai-4g.yaml
2019-06-12T19:39:35 INFO jox.NetworkSliceController nsi_controller.py:102 Adding the slice mosaic5g-oai-4g.yaml
2019-06-12T19:39:35 ERROR jox.es es.py:219 Error while getting the source of the index of slice_keys_oai-4g from elasticsearch: NotFoundError(404, 'index_not_found_exception', 'no such index [slice_keys_oai-4g]')
2019-06-12T19:39:36 ERROR jox.es es.py:219 Error while getting the source of the index of slice_monitor_oai-4g from elasticsearch: NotFoundError(404, 'index_not_found_exception', 'no such index [slice_monitor_oai-4g]')
2019-06-12T19:39:36 INFO jox.templateManager template_manager.py:128 Deleting the index oai-4g from elasticsearch if alredy exist
2019-06-12T19:39:36 DEBUG jox.templateManager template_manager.py:129 Deleting the index oai-4g from elasticsearch if alredy exist
2019-06-12T19:39:36 INFO jox.templateManager template_manager.py:132 Saving the index oai-4g to elasticsearch
2019-06-12T19:39:36 DEBUG jox.templateManager template_manager.py:133 Saving the index oai-4g to elasticsearch
2019-06-12T19:39:36 ERROR jox.es es.py:219 Error while getting the source of the index of oai-4g from elasticsearch: NotFoundError(404, 'index_not_found_exception', 'no such index [oai-4g]')
2019-06-12T19:39:36 INFO jox.templateManager template_manager.py:155 Deleting the index nssi_1 from elasticsearch if already exist
2019-06-12T19:39:36 DEBUG jox.templateManager template_manager.py:156 Deleting the index nssi_1 from elasticsearch if already exist
2019-06-12T19:39:36 INFO jox.templateManager template_manager.py:161 Saving the index nssi_1 to elasticsearch
2019-06-12T19:39:36 DEBUG jox.templateManager template_manager.py:162 Saving the index nssi_1 to elasticsearch
2019-06-12T19:39:36 ERROR jox.es es.py:219 Error while getting the source of the index of nssi_1 from elasticsearch: NotFoundError(404, 'index_not_found_exception', 'no such index [nssi_1]')
2019-06-12T19:39:37 INFO jox.templateManager template_manager.py:155 Deleting the index nssi_2 from elasticsearch if already exist
2019-06-12T19:39:37 DEBUG jox.templateManager template_manager.py:156 Deleting the index nssi_2 from elasticsearch if already exist
2019-06-12T19:39:37 INFO jox.templateManager template_manager.py:161 Saving the index nssi_2 to elasticsearch
2019-06-12T19:39:37 DEBUG jox.templateManager template_manager.py:162 Saving the index nssi_2 to elasticsearch
2019-06-12T19:39:37 ERROR jox.es es.py:219 Error while getting the source of the index of nssi_2 from elasticsearch: NotFoundError(404, 'index_not_found_exception', 'no such index [nssi_2]')
2019-06-12T19:39:37 INFO jox.templateManager template_manager.py:155 Deleting the index nssi_3 from elasticsearch if already exist
2019-06-12T19:39:37 DEBUG jox.templateManager template_manager.py:156 Deleting the index nssi_3 from elasticsearch if already exist
2019-06-12T19:39:37 INFO jox.templateManager template_manager.py:161 Saving the index nssi_3 to elasticsearch
2019-06-12T19:39:37 DEBUG jox.templateManager template_manager.py:162 Saving the index nssi_3 to elasticsearch
2019-06-12T19:39:37 ERROR jox.es es.py:219 Error while getting the source of the index of nssi_3 from elasticsearch: NotFoundError(404, 'index_not_found_exception', 'no such index [nssi_3]')
2019-06-12T19:39:37 ERROR jox.es es.py:219 Error while getting the source of the index of subslice_monitor_nssi_1 from elasticsearch: NotFoundError(404, 'index_not_found_exception', 'no such index [subslice_monitor_nssi_1]')
2019-06-12T19:39:38 INFO jox.jslice slice.py:105 trying to add the subslice nssi_1
2019-06-12T19:39:38 INFO jox.subslice subslice.py:118 add the models on which the services of the  NSSI nssi_1 will be deployed
2019-06-12T19:39:39 INFO jox.jmodel model.py:143 The juju model nymphe-edu:default is configured
2019-06-12T19:39:39 DEBUG jox.jmodel model.py:144 The ID of the juju model nymphe-edu:default is f5240599-4e6f-44ff-8951-2419243bf1f6
2019-06-12T19:39:39 INFO jox.JCloud cloud.py:170 Connecting to controller
2019-06-12T19:39:39 INFO jox.JCloud cloud.py:170 Connecting to controller
2019-06-12T19:39:39 INFO jox.JCloud cloud.py:170 Connecting to controller
2019-06-12T19:39:39 INFO jox.JCloud cloud.py:170 Connecting to controller
2019-06-12T19:39:39 INFO jox.RO.rsourceController resource_controller.py:342 Adding machine for the application mysql
2019-06-12T19:39:39 INFO jox.RO.rsourceController resource_controller.py:441 Adding kvm machine whose id is: machine_name= 1:kvm, mid_ro = VDU_mysql
2019-06-12T19:39:39 DEBUG jox.KvmDriver.default vimdriver.py:723 0 time trying to connect to juju model:nymphe-edu:admin/default to add lxc machine
2019-06-12T19:39:39 DEBUG jox.KvmDriver.default vimdriver.py:726 Successful connection to the model nymphe-edu:admin/default 
2019-06-12T19:39:40 DEBUG jox.KvmDriver.default vimdriver.py:832 The machine VDU_mysql with juju id 48 is already added to juju model
2019-06-12T19:39:40 DEBUG jox.RO.rsourceController resource_controller.py:453 ('mysql', ' kvm machine added')
2019-06-12T19:39:40 INFO jox.RO.rsourceController resource_controller.py:342 Adding machine for the application flexran
2019-06-12T19:39:40 INFO jox.RO.rsourceController resource_controller.py:441 Adding kvm machine whose id is: machine_name= 2:kvm, mid_ro = VDU_flexran
2019-06-12T19:39:40 DEBUG jox.KvmDriver.default vimdriver.py:723 0 time trying to connect to juju model:nymphe-edu:admin/default to add lxc machine
2019-06-12T19:39:40 DEBUG jox.KvmDriver.default vimdriver.py:726 Successful connection to the model nymphe-edu:admin/default 
2019-06-12T19:39:40 DEBUG jox.KvmDriver.default vimdriver.py:832 The machine VDU_flexran with juju id 49 is already added to juju model
2019-06-12T19:39:40 DEBUG jox.RO.rsourceController resource_controller.py:453 ('flexran', ' kvm machine added')
2019-06-12T19:39:40 INFO jox.RO.rsourceController resource_controller.py:342 Adding machine for the application oai-hss
2019-06-12T19:39:40 INFO jox.RO.rsourceController resource_controller.py:441 Adding kvm machine whose id is: machine_name= 3:kvm, mid_ro = VDU_oai_hss
2019-06-12T19:39:40 DEBUG jox.KvmDriver.default vimdriver.py:723 0 time trying to connect to juju model:nymphe-edu:admin/default to add lxc machine
2019-06-12T19:39:40 DEBUG jox.KvmDriver.default vimdriver.py:726 Successful connection to the model nymphe-edu:admin/default 
2019-06-12T19:39:41 DEBUG jox.KvmDriver.default vimdriver.py:832 The machine VDU_oai_hss with juju id 51 is already added to juju model
2019-06-12T19:39:41 DEBUG jox.RO.rsourceController resource_controller.py:453 ('oai-hss', ' kvm machine added')
2019-06-12T19:39:41 DEBUG jox.RO.rsourceController resource_controller.py:504 Waiting for the machines to start to deploy the services
2019-06-12T19:39:41 ERROR jox.nssiController nssi_controller.py:132 module 'src.common.config.gv' has no attribute 'PHY_2'
2019-06-12T19:39:41 ERROR jox.nssiController nssi_controller.py:133 Traceback (most recent call last):
  File "/home/psoque/JoX/jox-plugin/src/core/nso/nssi/nssi_controller.py", line 127, in add_subslice
    new_slice.build_jsubslice(subslice_config, self.resourceController, list_all_jujuModel_attachedWatcher)
  File "/home/psoque/JoX/jox-plugin/src/core/nso/nssi/subslice.py", line 121, in build_jsubslice
    raise ex
  File "/home/psoque/JoX/jox-plugin/src/core/nso/nssi/subslice.py", line 119, in build_jsubslice
    self.add_models(subslice_config, resourceController, list_all_jujuModel_attachedWatcher)
  File "/home/psoque/JoX/jox-plugin/src/core/nso/nssi/subslice.py", line 241, in add_models
    raise ex
  File "/home/psoque/JoX/jox-plugin/src/core/nso/nssi/subslice.py", line 234, in add_models
    self.add_model(subslice_config, resourceController, juju_cloud, juju_model, list_all_jujuModel_attachedWatcher)
  File "/home/psoque/JoX/jox-plugin/src/core/nso/nssi/subslice.py", line 265, in add_model
    raise ex
  File "/home/psoque/JoX/jox-plugin/src/core/nso/nssi/subslice.py", line 262, in add_model
    resourceController, juju_cloud, juju_model, Watcher_attached, self.subslice_machines)
  File "/home/psoque/JoX/jox-plugin/src/core/nso/nssi/model.py", line 166, in build_and_deploy
    raise ex
  File "/home/psoque/JoX/jox-plugin/src/core/nso/nssi/model.py", line 150, in build_and_deploy
    list_machine = resourceController.add_machines(self.machines_config, self.services_config, self.subslice_name, self.model_name, self.cloud_name, self.slice_name)
  File "/home/psoque/JoX/jox-plugin/src/core/ro/resource_controller.py", line 533, in add_machines
    raise ex
  File "/home/psoque/JoX/jox-plugin/src/core/ro/resource_controller.py", line 523, in add_machines
    if [(self.gv.PHY == pop_type or self.gv.PHY_2 == pop_type) for pop_type in machine_types]:
  File "/home/psoque/JoX/jox-plugin/src/core/ro/resource_controller.py", line 523, in <listcomp>
    if [(self.gv.PHY == pop_type or self.gv.PHY_2 == pop_type) for pop_type in machine_types]:
AttributeError: module 'src.common.config.gv' has no attribute 'PHY_2'

2019-06-12T19:39:41 INFO jox.jslice slice.py:111 the subslice nssi_1 is NOT added. Exit...
2019-06-12T19:39:41 ERROR jox.es es.py:219 Error while getting the source of the index of subslice_monitor_nssi_2 from elasticsearch: NotFoundError(404, 'index_not_found_exception', 'no such index [subslice_monitor_nssi_2]')
2019-06-12T19:39:41 INFO jox.jslice slice.py:105 trying to add the subslice nssi_2
2019-06-12T19:39:41 INFO jox.subslice subslice.py:118 add the models on which the services of the  NSSI nssi_2 will be deployed
2019-06-12T19:39:42 INFO jox.jmodel model.py:143 The juju model nymphe-edu:default is configured
2019-06-12T19:39:42 DEBUG jox.jmodel model.py:144 The ID of the juju model nymphe-edu:default is f5240599-4e6f-44ff-8951-2419243bf1f6
2019-06-12T19:39:42 INFO jox.RO.rsourceController resource_controller.py:342 Adding machine for the application oai-mme
2019-06-12T19:39:42 INFO jox.RO.rsourceController resource_controller.py:441 Adding kvm machine whose id is: machine_name= 4:kvm, mid_ro = VDU_oai-mme
2019-06-12T19:39:42 DEBUG jox.KvmDriver.default vimdriver.py:723 0 time trying to connect to juju model:nymphe-edu:admin/default to add lxc machine
2019-06-12T19:39:42 DEBUG jox.KvmDriver.default vimdriver.py:726 Successful connection to the model nymphe-edu:admin/default 
2019-06-12T19:39:42 DEBUG jox.KvmDriver.default vimdriver.py:832 The machine VDU_oai-mme with juju id 47 is already added to juju model
2019-06-12T19:39:42 DEBUG jox.RO.rsourceController resource_controller.py:453 ('oai-mme', ' kvm machine added')
2019-06-12T19:39:42 INFO jox.RO.rsourceController resource_controller.py:342 Adding machine for the application oai-spgw
2019-06-12T19:39:42 INFO jox.RO.rsourceController resource_controller.py:441 Adding kvm machine whose id is: machine_name= 5:kvm, mid_ro = VDU_oai-spgw
2019-06-12T19:39:42 DEBUG jox.KvmDriver.default vimdriver.py:723 0 time trying to connect to juju model:nymphe-edu:admin/default to add lxc machine
2019-06-12T19:39:42 DEBUG jox.KvmDriver.default vimdriver.py:726 Successful connection to the model nymphe-edu:admin/default 
2019-06-12T19:39:43 DEBUG jox.KvmDriver.default vimdriver.py:832 The machine VDU_oai-spgw with juju id 50 is already added to juju model
2019-06-12T19:39:43 DEBUG jox.RO.rsourceController resource_controller.py:453 ('oai-spgw', ' kvm machine added')
2019-06-12T19:39:43 DEBUG jox.RO.rsourceController resource_controller.py:504 Waiting for the machines to start to deploy the services
2019-06-12T19:39:43 ERROR jox.nssiController nssi_controller.py:132 module 'src.common.config.gv' has no attribute 'PHY_2'
2019-06-12T19:39:43 ERROR jox.nssiController nssi_controller.py:133 Traceback (most recent call last):
  File "/home/psoque/JoX/jox-plugin/src/core/nso/nssi/nssi_controller.py", line 127, in add_subslice
    new_slice.build_jsubslice(subslice_config, self.resourceController, list_all_jujuModel_attachedWatcher)
  File "/home/psoque/JoX/jox-plugin/src/core/nso/nssi/subslice.py", line 121, in build_jsubslice
    raise ex
  File "/home/psoque/JoX/jox-plugin/src/core/nso/nssi/subslice.py", line 119, in build_jsubslice
    self.add_models(subslice_config, resourceController, list_all_jujuModel_attachedWatcher)
  File "/home/psoque/JoX/jox-plugin/src/core/nso/nssi/subslice.py", line 241, in add_models
    raise ex
  File "/home/psoque/JoX/jox-plugin/src/core/nso/nssi/subslice.py", line 234, in add_models
    self.add_model(subslice_config, resourceController, juju_cloud, juju_model, list_all_jujuModel_attachedWatcher)
  File "/home/psoque/JoX/jox-plugin/src/core/nso/nssi/subslice.py", line 265, in add_model
    raise ex
  File "/home/psoque/JoX/jox-plugin/src/core/nso/nssi/subslice.py", line 262, in add_model
    resourceController, juju_cloud, juju_model, Watcher_attached, self.subslice_machines)
  File "/home/psoque/JoX/jox-plugin/src/core/nso/nssi/model.py", line 166, in build_and_deploy
    raise ex
  File "/home/psoque/JoX/jox-plugin/src/core/nso/nssi/model.py", line 150, in build_and_deploy
    list_machine = resourceController.add_machines(self.machines_config, self.services_config, self.subslice_name, self.model_name, self.cloud_name, self.slice_name)
  File "/home/psoque/JoX/jox-plugin/src/core/ro/resource_controller.py", line 533, in add_machines
    raise ex
  File "/home/psoque/JoX/jox-plugin/src/core/ro/resource_controller.py", line 523, in add_machines
    if [(self.gv.PHY == pop_type or self.gv.PHY_2 == pop_type) for pop_type in machine_types]:
  File "/home/psoque/JoX/jox-plugin/src/core/ro/resource_controller.py", line 523, in <listcomp>
    if [(self.gv.PHY == pop_type or self.gv.PHY_2 == pop_type) for pop_type in machine_types]:
AttributeError: module 'src.common.config.gv' has no attribute 'PHY_2'

2019-06-12T19:39:43 INFO jox.jslice slice.py:111 the subslice nssi_2 is NOT added. Exit...
2019-06-12T19:39:43 ERROR jox.es es.py:219 Error while getting the source of the index of subslice_monitor_nssi_3 from elasticsearch: NotFoundError(404, 'index_not_found_exception', 'no such index [subslice_monitor_nssi_3]')
2019-06-12T19:39:43 INFO jox.jslice slice.py:105 trying to add the subslice nssi_3
2019-06-12T19:39:43 INFO jox.subslice subslice.py:118 add the models on which the services of the  NSSI nssi_3 will be deployed
2019-06-12T19:39:44 INFO jox.jmodel model.py:143 The juju model nymphe-edu:default is configured
2019-06-12T19:39:44 DEBUG jox.jmodel model.py:144 The ID of the juju model nymphe-edu:default is f5240599-4e6f-44ff-8951-2419243bf1f6
2019-06-12T19:39:44 INFO jox.RO.rsourceController resource_controller.py:342 Adding machine for the application oai-ran
2019-06-12T19:39:44 INFO jox.RO.rsourceController resource_controller.py:486 Adding physical machine whose id is: machine_name= 6:phy, mid_ro = VDU_oai-ran
2019-06-12T19:39:44 DEBUG jox.PhyDriver.zone-1-phy vimdriver.py:1079 0 time trying to connect to juju model:nymphe-edu:admin/default to add lxc machine
2019-06-12T19:39:44 DEBUG jox.PhyDriver.zone-1-phy vimdriver.py:1082 Successful connection to the model nymphe-edu:admin/default 
2019-06-12T19:39:44 DEBUG jox.RO.rsourceController resource_controller.py:498 ('oai-ran', ' kvm machine added')
2019-06-12T19:39:44 DEBUG jox.RO.rsourceController resource_controller.py:504 Waiting for the machines to start to deploy the services
2019-06-12T19:39:44 INFO jox.jmodel model.py:202 Deploying the application oai-ran
2019-06-12 19:39:53,714 - jox.JujuModelServiceController - INFO - The servce oai-ran is deployed
2019-06-12 19:39:53,714 - jox.JujuModelServiceController - INFO - The servce oai-ran is deployed
2019-06-12 19:39:53,714 - jox.JujuModelServiceController - INFO - The servce oai-ran is deployed
2019-06-12T19:39:53 INFO jox.JujuModelServiceController juju2_plugin.py:285 The servce oai-ran is deployed
2019-06-12 19:39:53,716 - jox.JujuModelServiceController - DEBUG - The servce oai-ran is deployed
2019-06-12 19:39:53,716 - jox.JujuModelServiceController - DEBUG - The servce oai-ran is deployed
2019-06-12 19:39:53,716 - jox.JujuModelServiceController - DEBUG - The servce oai-ran is deployed
2019-06-12T19:39:53 DEBUG jox.JujuModelServiceController juju2_plugin.py:286 The servce oai-ran is deployed
2019-06-12T19:39:53 INFO jox.nssiController nssi_controller.py:129 Slice nssi_3 was added
2019-06-12T19:39:53 INFO jox.jslice slice.py:108 the subslice nssi_3 is successfully added
2019-06-12T19:39:53 INFO jox.jslice slice.py:115 adding entre-nssi relations
2019-06-12T19:39:53 INFO jox.jslice slice.py:81 getting the inter relations of the slice oai-4g
2019-06-12T19:39:53 DEBUG jox.jslice slice.py:156 adding relation between oai-hss from the nssi nssi_1 and oai-mme from the nssi nssi_2
2019-06-12T19:40:03 DEBUG jox.jslice slice.py:156 adding relation between oai-ran from the nssi nssi_3 and flexran from the nssi nssi_1
2019-06-12T19:40:15 INFO jox jox.py:233 Configure elastcisearch if enabled
2019-06-12T19:40:15 INFO jox jox.py:244 elasticsearch is now running
2019-06-12T19:40:15 INFO jox jox.py:247 Save JOX configuration to elasticsearch
2019-06-12T19:40:15 INFO jox jox.py:250 Delete the index of jox config from elasticsearch if exist
2019-06-12T19:40:15 INFO jox jox.py:253 Sending JOX configuration file to elasticsearch
2019-06-12T19:40:15 ERROR jox.es es.py:219 Error while getting the source of the index of jox_config from elasticsearch: NotFoundError(404, 'index_not_found_exception', 'no such index [jox_config]')
2019-06-12T19:40:15 INFO jox jox.py:257 The data of the index jox_config is addedd to elasticsearch
2019-06-12T19:40:15 INFO jox jox.py:266 Create resource Controller
2019-06-12T19:40:15 INFO jox jox.py:274 registering the following POP of type kvm as specified in jox_config: {'pop-name': 'default', 'ssh-password': '', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', 'managed-by': 'kvm', 'zone': '1', 'ssh-user': '', 'domain': '192.168.122.0/24', '_Comment_3': 'ssh config', 'scope': 'private', 'ssh-private-key': '', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty'}
2019-06-12T19:40:15 DEBUG jox jox.py:391 The kvm pop with the following config is successfully regiestered: {'pop-name': 'default', 'ssh-password': '', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', 'managed-by': 'kvm', 'zone': '1', 'ssh-user': '', 'domain': '192.168.122.0/24', '_Comment_3': 'ssh config', 'scope': 'private', 'ssh-private-key': '', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty'}
2019-06-12T19:40:15 INFO jox jox.py:274 registering the following POP of type lxc as specified in jox_config: {'pop-name': 'default', 'ssh-password': '', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', 'managed-by': 'lxd', 'zone': '1', 'ssh-user': '', 'domain': '10.146.98.0/24', '_Comment_3': 'ssh config', 'scope': 'private', 'ssh-private-key': '', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty'}
2019-06-12T19:40:15 DEBUG jox jox.py:389 The following error raised while registering lxc pop with the config {'pop-name': 'default', 'ssh-password': '', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', 'managed-by': 'lxd', 'zone': '1', 'ssh-user': '', 'domain': '10.146.98.0/24', '_Comment_3': 'ssh config', 'scope': 'private', 'ssh-private-key': '', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty'}: 
 'lxc-anchor'
2019-06-12T19:40:15 INFO jox jox.py:274 registering the following POP of type phy as specified in jox_config: {'pop-name': 'zone-1-phy', 'ssh-password': '', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', 'managed-by': 'none', 'zone': '1', 'ssh-user': '', 'domain': '192.168.12.0/24', '_Comment_3': 'ssh config', 'scope': 'private', 'ssh-private-key': '', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty'}
2019-06-12T19:40:15 DEBUG jox jox.py:391 The phy pop with the following config is successfully regiestered: {'pop-name': 'zone-1-phy', 'ssh-password': '', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', 'managed-by': 'none', 'zone': '1', 'ssh-user': '', 'domain': '192.168.12.0/24', '_Comment_3': 'ssh config', 'scope': 'private', 'ssh-private-key': '', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty'}
2019-06-12T19:40:15 INFO jox jox.py:274 registering the following POP of type phy as specified in jox_config: {'pop-name': 'zone-2-phy', 'ssh-password': '', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', 'managed-by': 'none', 'zone': '1', 'ssh-user': '', 'domain': '172.24.0.0/16', '_Comment_3': 'ssh config', 'scope': 'private', 'ssh-private-key': '', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty'}
2019-06-12T19:40:15 DEBUG jox jox.py:391 The phy pop with the following config is successfully regiestered: {'pop-name': 'zone-2-phy', 'ssh-password': '', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', 'managed-by': 'none', 'zone': '1', 'ssh-user': '', 'domain': '172.24.0.0/16', '_Comment_3': 'ssh config', 'scope': 'private', 'ssh-private-key': '', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty'}
2019-06-12T19:40:15 INFO jox jox.py:274 registering the following POP of type phy as specified in jox_config: {'pop-name': 'zone-3-phy', 'ssh-password': '', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', 'managed-by': 'none', 'zone': '1', 'ssh-user': '', 'domain': '172.24.10.0/24', '_Comment_3': 'ssh config', 'scope': 'private', 'ssh-private-key': '', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty'}
2019-06-12T19:40:15 DEBUG jox jox.py:391 The phy pop with the following config is successfully regiestered: {'pop-name': 'zone-3-phy', 'ssh-password': '', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', 'managed-by': 'none', 'zone': '1', 'ssh-user': '', 'domain': '172.24.10.0/24', '_Comment_3': 'ssh config', 'scope': 'private', 'ssh-private-key': '', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty'}
2019-06-12T19:40:15 INFO jox jox.py:274 registering the following POP of type phy as specified in jox_config: {'pop-name': 'zone-4-phy', 'ssh-password': '', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', 'managed-by': 'none', 'zone': '1', 'ssh-user': '', 'domain': '172.24.11.0/24', '_Comment_3': 'ssh config', 'scope': 'private', 'ssh-private-key': '', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty'}
2019-06-12T19:40:15 DEBUG jox jox.py:391 The phy pop with the following config is successfully regiestered: {'pop-name': 'zone-4-phy', 'ssh-password': '', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', 'managed-by': 'none', 'zone': '1', 'ssh-user': '', 'domain': '172.24.11.0/24', '_Comment_3': 'ssh config', 'scope': 'private', 'ssh-private-key': '', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty'}
2019-06-12T19:40:15 INFO jox jox.py:281 Clouds Controller is successfully loaded!
2019-06-12T19:40:15 DEBUG jox.RO.rsourceController resource_controller.py:539 jcloud localhost was added to jox
2019-06-12T19:40:15 DEBUG jox jox.py:405 The following cloud is successfully  added
2019-06-12T19:40:15 DEBUG jox.RO.rsourceController resource_controller.py:539 jcloud nymphe-edu was added to jox
2019-06-12T19:40:15 DEBUG jox jox.py:405 The following cloud is successfully  added
2019-06-12T19:40:15 INFO jox jox.py:293 All clouds were successfully added!
2019-06-12T19:40:15 INFO jox jox.py:297 Creating slice controller
2019-06-12T19:40:15 INFO jox.NetworkSliceController nsi_controller.py:41 Initial configuration of network slice controller
2019-06-12T19:40:15 INFO jox.NetworkSliceController nsi_controller.py:62 The resource controller is built
2019-06-12T19:40:15 INFO jox jox.py:301 Creating sub-slice controller
2019-06-12T19:40:15 INFO jox jox.py:309 Slices Controller was successfully loaded!
2019-06-12T19:40:15 INFO jox jox.py:314 JOX loading time: 0.2998476028442383 
2019-06-12T19:40:15 INFO jox jox.py:324 JOX Web API loaded!
2019-06-12T19:40:18 INFO jox jox.py:410 Adding the slice mosaic5g-oai-4g.yaml
2019-06-12T19:40:18 INFO jox.NetworkSliceController nsi_controller.py:102 Adding the slice mosaic5g-oai-4g.yaml
2019-06-12T19:40:18 ERROR jox.es es.py:219 Error while getting the source of the index of slice_keys_oai-4g from elasticsearch: NotFoundError(404, 'index_not_found_exception', 'no such index [slice_keys_oai-4g]')
2019-06-12T19:40:18 ERROR jox.es es.py:219 Error while getting the source of the index of slice_monitor_oai-4g from elasticsearch: NotFoundError(404, 'index_not_found_exception', 'no such index [slice_monitor_oai-4g]')
2019-06-12T19:40:19 INFO jox.templateManager template_manager.py:128 Deleting the index oai-4g from elasticsearch if alredy exist
2019-06-12T19:40:19 DEBUG jox.templateManager template_manager.py:129 Deleting the index oai-4g from elasticsearch if alredy exist
2019-06-12T19:40:19 INFO jox.templateManager template_manager.py:132 Saving the index oai-4g to elasticsearch
2019-06-12T19:40:19 DEBUG jox.templateManager template_manager.py:133 Saving the index oai-4g to elasticsearch
2019-06-12T19:40:19 ERROR jox.es es.py:219 Error while getting the source of the index of oai-4g from elasticsearch: NotFoundError(404, 'index_not_found_exception', 'no such index [oai-4g]')
2019-06-12T19:40:19 INFO jox.templateManager template_manager.py:155 Deleting the index nssi_1 from elasticsearch if already exist
2019-06-12T19:40:19 DEBUG jox.templateManager template_manager.py:156 Deleting the index nssi_1 from elasticsearch if already exist
2019-06-12T19:40:19 INFO jox.templateManager template_manager.py:161 Saving the index nssi_1 to elasticsearch
2019-06-12T19:40:19 DEBUG jox.templateManager template_manager.py:162 Saving the index nssi_1 to elasticsearch
2019-06-12T19:40:19 ERROR jox.es es.py:219 Error while getting the source of the index of nssi_1 from elasticsearch: NotFoundError(404, 'index_not_found_exception', 'no such index [nssi_1]')
2019-06-12T19:40:19 INFO jox.templateManager template_manager.py:155 Deleting the index nssi_2 from elasticsearch if already exist
2019-06-12T19:40:19 DEBUG jox.templateManager template_manager.py:156 Deleting the index nssi_2 from elasticsearch if already exist
2019-06-12T19:40:19 INFO jox.templateManager template_manager.py:161 Saving the index nssi_2 to elasticsearch
2019-06-12T19:40:19 DEBUG jox.templateManager template_manager.py:162 Saving the index nssi_2 to elasticsearch
2019-06-12T19:40:19 ERROR jox.es es.py:219 Error while getting the source of the index of nssi_2 from elasticsearch: NotFoundError(404, 'index_not_found_exception', 'no such index [nssi_2]')
2019-06-12T19:40:20 INFO jox.templateManager template_manager.py:155 Deleting the index nssi_3 from elasticsearch if already exist
2019-06-12T19:40:20 DEBUG jox.templateManager template_manager.py:156 Deleting the index nssi_3 from elasticsearch if already exist
2019-06-12T19:40:20 INFO jox.templateManager template_manager.py:161 Saving the index nssi_3 to elasticsearch
2019-06-12T19:40:20 DEBUG jox.templateManager template_manager.py:162 Saving the index nssi_3 to elasticsearch
2019-06-12T19:40:20 ERROR jox.es es.py:219 Error while getting the source of the index of nssi_3 from elasticsearch: NotFoundError(404, 'index_not_found_exception', 'no such index [nssi_3]')
2019-06-12T19:40:20 ERROR jox.es es.py:219 Error while getting the source of the index of subslice_monitor_nssi_1 from elasticsearch: NotFoundError(404, 'index_not_found_exception', 'no such index [subslice_monitor_nssi_1]')
2019-06-12T19:40:20 INFO jox.jslice slice.py:105 trying to add the subslice nssi_1
2019-06-12T19:40:20 INFO jox.subslice subslice.py:118 add the models on which the services of the  NSSI nssi_1 will be deployed
2019-06-12T19:40:21 INFO jox.jmodel model.py:143 The juju model nymphe-edu:default is configured
2019-06-12T19:40:21 DEBUG jox.jmodel model.py:144 The ID of the juju model nymphe-edu:default is f5240599-4e6f-44ff-8951-2419243bf1f6
2019-06-12T19:40:21 INFO jox.JCloud cloud.py:170 Connecting to controller
2019-06-12T19:40:21 INFO jox.JCloud cloud.py:170 Connecting to controller
2019-06-12T19:40:21 INFO jox.JCloud cloud.py:170 Connecting to controller
2019-06-12T19:40:21 INFO jox.JCloud cloud.py:170 Connecting to controller
2019-06-12T19:40:21 INFO jox.RO.rsourceController resource_controller.py:342 Adding machine for the application mysql
2019-06-12T19:40:21 INFO jox.RO.rsourceController resource_controller.py:441 Adding kvm machine whose id is: machine_name= 1:kvm, mid_ro = VDU_mysql
2019-06-12T19:40:21 DEBUG jox.KvmDriver.default vimdriver.py:723 0 time trying to connect to juju model:nymphe-edu:admin/default to add lxc machine
2019-06-12T19:40:22 DEBUG jox.KvmDriver.default vimdriver.py:726 Successful connection to the model nymphe-edu:admin/default 
2019-06-12T19:40:22 DEBUG jox.KvmDriver.default vimdriver.py:832 The machine VDU_mysql with juju id 48 is already added to juju model
2019-06-12T19:40:22 DEBUG jox.RO.rsourceController resource_controller.py:453 ('mysql', ' kvm machine added')
2019-06-12T19:40:22 INFO jox.RO.rsourceController resource_controller.py:342 Adding machine for the application flexran
2019-06-12T19:40:22 INFO jox.RO.rsourceController resource_controller.py:441 Adding kvm machine whose id is: machine_name= 2:kvm, mid_ro = VDU_flexran
2019-06-12T19:40:22 DEBUG jox.KvmDriver.default vimdriver.py:723 0 time trying to connect to juju model:nymphe-edu:admin/default to add lxc machine
2019-06-12T19:40:22 DEBUG jox.KvmDriver.default vimdriver.py:726 Successful connection to the model nymphe-edu:admin/default 
2019-06-12T19:40:23 DEBUG jox.KvmDriver.default vimdriver.py:832 The machine VDU_flexran with juju id 49 is already added to juju model
2019-06-12T19:40:23 DEBUG jox.RO.rsourceController resource_controller.py:453 ('flexran', ' kvm machine added')
2019-06-12T19:40:23 INFO jox.RO.rsourceController resource_controller.py:342 Adding machine for the application oai-hss
2019-06-12T19:40:23 INFO jox.RO.rsourceController resource_controller.py:441 Adding kvm machine whose id is: machine_name= 3:kvm, mid_ro = VDU_oai_hss
2019-06-12T19:40:23 DEBUG jox.KvmDriver.default vimdriver.py:723 0 time trying to connect to juju model:nymphe-edu:admin/default to add lxc machine
2019-06-12T19:40:23 DEBUG jox.KvmDriver.default vimdriver.py:726 Successful connection to the model nymphe-edu:admin/default 
2019-06-12T19:40:23 DEBUG jox.KvmDriver.default vimdriver.py:832 The machine VDU_oai_hss with juju id 51 is already added to juju model
2019-06-12T19:40:23 DEBUG jox.RO.rsourceController resource_controller.py:453 ('oai-hss', ' kvm machine added')
2019-06-12T19:40:23 DEBUG jox.RO.rsourceController resource_controller.py:504 Waiting for the machines to start to deploy the services
2019-06-12T19:40:23 ERROR jox.nssiController nssi_controller.py:132 module 'src.common.config.gv' has no attribute 'PHY_2'
2019-06-12T19:40:23 ERROR jox.nssiController nssi_controller.py:133 Traceback (most recent call last):
  File "/home/psoque/JoX/jox-plugin/src/core/nso/nssi/nssi_controller.py", line 127, in add_subslice
    new_slice.build_jsubslice(subslice_config, self.resourceController, list_all_jujuModel_attachedWatcher)
  File "/home/psoque/JoX/jox-plugin/src/core/nso/nssi/subslice.py", line 121, in build_jsubslice
    raise ex
  File "/home/psoque/JoX/jox-plugin/src/core/nso/nssi/subslice.py", line 119, in build_jsubslice
    self.add_models(subslice_config, resourceController, list_all_jujuModel_attachedWatcher)
  File "/home/psoque/JoX/jox-plugin/src/core/nso/nssi/subslice.py", line 241, in add_models
    raise ex
  File "/home/psoque/JoX/jox-plugin/src/core/nso/nssi/subslice.py", line 234, in add_models
    self.add_model(subslice_config, resourceController, juju_cloud, juju_model, list_all_jujuModel_attachedWatcher)
  File "/home/psoque/JoX/jox-plugin/src/core/nso/nssi/subslice.py", line 265, in add_model
    raise ex
  File "/home/psoque/JoX/jox-plugin/src/core/nso/nssi/subslice.py", line 262, in add_model
    resourceController, juju_cloud, juju_model, Watcher_attached, self.subslice_machines)
  File "/home/psoque/JoX/jox-plugin/src/core/nso/nssi/model.py", line 166, in build_and_deploy
    raise ex
  File "/home/psoque/JoX/jox-plugin/src/core/nso/nssi/model.py", line 150, in build_and_deploy
    list_machine = resourceController.add_machines(self.machines_config, self.services_config, self.subslice_name, self.model_name, self.cloud_name, self.slice_name)
  File "/home/psoque/JoX/jox-plugin/src/core/ro/resource_controller.py", line 533, in add_machines
    raise ex
  File "/home/psoque/JoX/jox-plugin/src/core/ro/resource_controller.py", line 523, in add_machines
    if [(self.gv.PHY == pop_type or self.gv.PHY_2 == pop_type) for pop_type in machine_types]:
  File "/home/psoque/JoX/jox-plugin/src/core/ro/resource_controller.py", line 523, in <listcomp>
    if [(self.gv.PHY == pop_type or self.gv.PHY_2 == pop_type) for pop_type in machine_types]:
AttributeError: module 'src.common.config.gv' has no attribute 'PHY_2'

2019-06-12T19:40:23 INFO jox.jslice slice.py:111 the subslice nssi_1 is NOT added. Exit...
2019-06-12T19:40:23 ERROR jox.es es.py:219 Error while getting the source of the index of subslice_monitor_nssi_2 from elasticsearch: NotFoundError(404, 'index_not_found_exception', 'no such index [subslice_monitor_nssi_2]')
2019-06-12T19:40:24 INFO jox.jslice slice.py:105 trying to add the subslice nssi_2
2019-06-12T19:40:24 INFO jox.subslice subslice.py:118 add the models on which the services of the  NSSI nssi_2 will be deployed
2019-06-12T19:40:24 INFO jox.jmodel model.py:143 The juju model nymphe-edu:default is configured
2019-06-12T19:40:24 DEBUG jox.jmodel model.py:144 The ID of the juju model nymphe-edu:default is f5240599-4e6f-44ff-8951-2419243bf1f6
2019-06-12T19:40:24 INFO jox.RO.rsourceController resource_controller.py:342 Adding machine for the application oai-spgw
2019-06-12T19:40:24 INFO jox.RO.rsourceController resource_controller.py:441 Adding kvm machine whose id is: machine_name= 4:kvm, mid_ro = VDU_oai-spgw
2019-06-12T19:40:24 DEBUG jox.KvmDriver.default vimdriver.py:723 0 time trying to connect to juju model:nymphe-edu:admin/default to add lxc machine
2019-06-12T19:40:25 DEBUG jox.KvmDriver.default vimdriver.py:726 Successful connection to the model nymphe-edu:admin/default 
2019-06-12T19:40:25 DEBUG jox.KvmDriver.default vimdriver.py:832 The machine VDU_oai-spgw with juju id 50 is already added to juju model
2019-06-12T19:40:25 DEBUG jox.RO.rsourceController resource_controller.py:453 ('oai-spgw', ' kvm machine added')
2019-06-12T19:40:25 INFO jox.RO.rsourceController resource_controller.py:342 Adding machine for the application oai-mme
2019-06-12T19:40:25 INFO jox.RO.rsourceController resource_controller.py:441 Adding kvm machine whose id is: machine_name= 5:kvm, mid_ro = VDU_oai-mme
2019-06-12T19:40:25 DEBUG jox.KvmDriver.default vimdriver.py:723 0 time trying to connect to juju model:nymphe-edu:admin/default to add lxc machine
2019-06-12T19:40:25 DEBUG jox.KvmDriver.default vimdriver.py:726 Successful connection to the model nymphe-edu:admin/default 
2019-06-12T19:40:25 DEBUG jox.KvmDriver.default vimdriver.py:832 The machine VDU_oai-mme with juju id 47 is already added to juju model
2019-06-12T19:40:25 DEBUG jox.RO.rsourceController resource_controller.py:453 ('oai-mme', ' kvm machine added')
2019-06-12T19:40:25 DEBUG jox.RO.rsourceController resource_controller.py:504 Waiting for the machines to start to deploy the services
2019-06-12T19:40:25 ERROR jox.nssiController nssi_controller.py:132 module 'src.common.config.gv' has no attribute 'PHY_2'
2019-06-12T19:40:25 ERROR jox.nssiController nssi_controller.py:133 Traceback (most recent call last):
  File "/home/psoque/JoX/jox-plugin/src/core/nso/nssi/nssi_controller.py", line 127, in add_subslice
    new_slice.build_jsubslice(subslice_config, self.resourceController, list_all_jujuModel_attachedWatcher)
  File "/home/psoque/JoX/jox-plugin/src/core/nso/nssi/subslice.py", line 121, in build_jsubslice
    raise ex
  File "/home/psoque/JoX/jox-plugin/src/core/nso/nssi/subslice.py", line 119, in build_jsubslice
    self.add_models(subslice_config, resourceController, list_all_jujuModel_attachedWatcher)
  File "/home/psoque/JoX/jox-plugin/src/core/nso/nssi/subslice.py", line 241, in add_models
    raise ex
  File "/home/psoque/JoX/jox-plugin/src/core/nso/nssi/subslice.py", line 234, in add_models
    self.add_model(subslice_config, resourceController, juju_cloud, juju_model, list_all_jujuModel_attachedWatcher)
  File "/home/psoque/JoX/jox-plugin/src/core/nso/nssi/subslice.py", line 265, in add_model
    raise ex
  File "/home/psoque/JoX/jox-plugin/src/core/nso/nssi/subslice.py", line 262, in add_model
    resourceController, juju_cloud, juju_model, Watcher_attached, self.subslice_machines)
  File "/home/psoque/JoX/jox-plugin/src/core/nso/nssi/model.py", line 166, in build_and_deploy
    raise ex
  File "/home/psoque/JoX/jox-plugin/src/core/nso/nssi/model.py", line 150, in build_and_deploy
    list_machine = resourceController.add_machines(self.machines_config, self.services_config, self.subslice_name, self.model_name, self.cloud_name, self.slice_name)
  File "/home/psoque/JoX/jox-plugin/src/core/ro/resource_controller.py", line 533, in add_machines
    raise ex
  File "/home/psoque/JoX/jox-plugin/src/core/ro/resource_controller.py", line 523, in add_machines
    if [(self.gv.PHY == pop_type or self.gv.PHY_2 == pop_type) for pop_type in machine_types]:
  File "/home/psoque/JoX/jox-plugin/src/core/ro/resource_controller.py", line 523, in <listcomp>
    if [(self.gv.PHY == pop_type or self.gv.PHY_2 == pop_type) for pop_type in machine_types]:
AttributeError: module 'src.common.config.gv' has no attribute 'PHY_2'

2019-06-12T19:40:25 INFO jox.jslice slice.py:111 the subslice nssi_2 is NOT added. Exit...
2019-06-12T19:40:25 ERROR jox.es es.py:219 Error while getting the source of the index of subslice_monitor_nssi_3 from elasticsearch: NotFoundError(404, 'index_not_found_exception', 'no such index [subslice_monitor_nssi_3]')
2019-06-12T19:40:26 INFO jox.jslice slice.py:105 trying to add the subslice nssi_3
2019-06-12T19:40:26 INFO jox.subslice subslice.py:118 add the models on which the services of the  NSSI nssi_3 will be deployed
2019-06-12T19:40:26 INFO jox.jmodel model.py:143 The juju model nymphe-edu:default is configured
2019-06-12T19:40:26 DEBUG jox.jmodel model.py:144 The ID of the juju model nymphe-edu:default is f5240599-4e6f-44ff-8951-2419243bf1f6
2019-06-12T19:40:26 INFO jox.RO.rsourceController resource_controller.py:342 Adding machine for the application oai-ran
2019-06-12T19:40:26 INFO jox.RO.rsourceController resource_controller.py:486 Adding physical machine whose id is: machine_name= 6:phy, mid_ro = VDU_oai-ran
2019-06-12T19:40:26 DEBUG jox.PhyDriver.zone-1-phy vimdriver.py:1079 0 time trying to connect to juju model:nymphe-edu:admin/default to add lxc machine
2019-06-12T19:40:27 DEBUG jox.PhyDriver.zone-1-phy vimdriver.py:1082 Successful connection to the model nymphe-edu:admin/default 
2019-06-12T19:40:27 DEBUG jox.RO.rsourceController resource_controller.py:498 ('oai-ran', ' kvm machine added')
2019-06-12T19:40:27 DEBUG jox.RO.rsourceController resource_controller.py:504 Waiting for the machines to start to deploy the services
2019-06-12T19:40:27 INFO jox.jmodel model.py:202 Deploying the application oai-ran
2019-06-12 19:40:36,266 - jox.JujuModelServiceController - INFO - The servce oai-ran is deployed
2019-06-12 19:40:36,266 - jox.JujuModelServiceController - INFO - The servce oai-ran is deployed
2019-06-12 19:40:36,266 - jox.JujuModelServiceController - INFO - The servce oai-ran is deployed
2019-06-12T19:40:36 INFO jox.JujuModelServiceController juju2_plugin.py:285 The servce oai-ran is deployed
2019-06-12 19:40:36,266 - jox.JujuModelServiceController - DEBUG - The servce oai-ran is deployed
2019-06-12 19:40:36,266 - jox.JujuModelServiceController - DEBUG - The servce oai-ran is deployed
2019-06-12 19:40:36,266 - jox.JujuModelServiceController - DEBUG - The servce oai-ran is deployed
2019-06-12T19:40:36 DEBUG jox.JujuModelServiceController juju2_plugin.py:286 The servce oai-ran is deployed
2019-06-12T19:40:36 INFO jox.nssiController nssi_controller.py:129 Slice nssi_3 was added
2019-06-12T19:40:36 INFO jox.jslice slice.py:108 the subslice nssi_3 is successfully added
2019-06-12T19:40:36 INFO jox.jslice slice.py:115 adding entre-nssi relations
2019-06-12T19:40:36 INFO jox.jslice slice.py:81 getting the inter relations of the slice oai-4g
2019-06-12T19:40:36 DEBUG jox.jslice slice.py:156 adding relation between oai-hss from the nssi nssi_1 and oai-mme from the nssi nssi_2
2019-06-12T19:40:46 DEBUG jox.jslice slice.py:156 adding relation between oai-ran from the nssi nssi_3 and flexran from the nssi nssi_1
2019-06-12T19:40:55 ERROR jox.monitor monitor.py:354 The key service_keys does not exist in the page slice_keys_oai-4g
2019-06-12T19:40:55 ERROR jox.monitor monitor.py:316 The key service_keys does not exist in the page slice_keys_oai-4g
2019-06-12T19:40:56 DEBUG jox.jslice slice.py:156 adding relation between oai-mme from the nssi nssi_2 and oai-ran from the nssi nssi_3
2019-06-12T19:41:07 INFO jox.NetworkSliceController nsi_controller.py:114 The slice mosaic5g-oai-4g.yaml is not deployed, due to the failure of adding at least one subslice
2019-06-12T19:41:07 DEBUG jox.NetworkSliceController nsi_controller.py:115 The slice mosaic5g-oai-4g.yaml is not deployed, due to the failure of adding at least one subslice
2019-06-12T19:41:07 ERROR jox.NetworkSliceController nsi_controller.py:116 The slice mosaic5g-oai-4g.yaml is not deployed, due to the failure of adding at least one subslice
2019-06-12T19:41:58 ERROR jox.monitor monitor.py:354 The key service_keys does not exist in the page slice_keys_oai-4g
2019-06-12T19:41:58 ERROR jox.monitor monitor.py:354 The key service_keys does not exist in the page slice_keys_oai-4g
2019-06-12T19:42:04 ERROR jox.monitor monitor.py:354 The key service_keys does not exist in the page slice_keys_oai-4g
2019-06-12T19:42:04 ERROR jox.monitor monitor.py:354 The key service_keys does not exist in the page slice_keys_oai-4g
2019-06-12T19:42:17 ERROR jox.monitor monitor.py:354 The key service_keys does not exist in the page slice_keys_oai-4g
2019-06-12T19:42:30 ERROR jox.monitor monitor.py:354 The key service_keys does not exist in the page slice_keys_oai-4g
2019-06-12T19:42:30 ERROR jox.monitor monitor.py:316 The key service_keys does not exist in the page slice_keys_oai-4g
2019-06-12T19:43:20 INFO jox jox.py:233 Configure elastcisearch if enabled
2019-06-12T19:43:20 INFO jox jox.py:244 elasticsearch is now running
2019-06-12T19:43:20 INFO jox jox.py:247 Save JOX configuration to elasticsearch
2019-06-12T19:43:20 INFO jox jox.py:250 Delete the index of jox config from elasticsearch if exist
2019-06-12T19:43:20 INFO jox jox.py:253 Sending JOX configuration file to elasticsearch
2019-06-12T19:43:20 ERROR jox.es es.py:219 Error while getting the source of the index of jox_config from elasticsearch: NotFoundError(404, 'index_not_found_exception', 'no such index [jox_config]')
2019-06-12T19:43:20 INFO jox jox.py:257 The data of the index jox_config is addedd to elasticsearch
2019-06-12T19:43:20 INFO jox jox.py:266 Create resource Controller
2019-06-12T19:43:20 INFO jox jox.py:274 registering the following POP of type lxc as specified in jox_config: {'zone': '1', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', 'managed-by': 'lxd', 'pop-name': 'default', 'ssh-user': '', 'scope': 'private', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty', 'ssh-private-key': '', '_Comment_3': 'ssh config', 'ssh-password': '', 'domain': '10.146.98.0/24'}
2019-06-12T19:43:20 DEBUG jox jox.py:389 The following error raised while registering lxc pop with the config {'zone': '1', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', 'managed-by': 'lxd', 'pop-name': 'default', 'ssh-user': '', 'scope': 'private', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty', 'ssh-private-key': '', '_Comment_3': 'ssh config', 'ssh-password': '', 'domain': '10.146.98.0/24'}: 
 'lxc-anchor'
2019-06-12T19:43:20 INFO jox jox.py:274 registering the following POP of type kvm as specified in jox_config: {'zone': '1', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', 'managed-by': 'kvm', 'pop-name': 'default', 'ssh-user': '', 'scope': 'private', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty', 'ssh-private-key': '', '_Comment_3': 'ssh config', 'ssh-password': '', 'domain': '192.168.122.0/24'}
2019-06-12T19:43:20 DEBUG jox jox.py:391 The kvm pop with the following config is successfully regiestered: {'zone': '1', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', 'managed-by': 'kvm', 'pop-name': 'default', 'ssh-user': '', 'scope': 'private', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty', 'ssh-private-key': '', '_Comment_3': 'ssh config', 'ssh-password': '', 'domain': '192.168.122.0/24'}
2019-06-12T19:43:20 INFO jox jox.py:274 registering the following POP of type phy as specified in jox_config: {'zone': '1', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', 'managed-by': 'none', 'pop-name': 'zone-1-phy', 'ssh-user': '', 'scope': 'private', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty', 'ssh-private-key': '', '_Comment_3': 'ssh config', 'ssh-password': '', 'domain': '192.168.12.0/24'}
2019-06-12T19:43:20 DEBUG jox jox.py:391 The phy pop with the following config is successfully regiestered: {'zone': '1', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', 'managed-by': 'none', 'pop-name': 'zone-1-phy', 'ssh-user': '', 'scope': 'private', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty', 'ssh-private-key': '', '_Comment_3': 'ssh config', 'ssh-password': '', 'domain': '192.168.12.0/24'}
2019-06-12T19:43:20 INFO jox jox.py:274 registering the following POP of type phy as specified in jox_config: {'zone': '1', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', 'managed-by': 'none', 'pop-name': 'zone-2-phy', 'ssh-user': '', 'scope': 'private', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty', 'ssh-private-key': '', '_Comment_3': 'ssh config', 'ssh-password': '', 'domain': '172.24.0.0/16'}
2019-06-12T19:43:20 DEBUG jox jox.py:391 The phy pop with the following config is successfully regiestered: {'zone': '1', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', 'managed-by': 'none', 'pop-name': 'zone-2-phy', 'ssh-user': '', 'scope': 'private', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty', 'ssh-private-key': '', '_Comment_3': 'ssh config', 'ssh-password': '', 'domain': '172.24.0.0/16'}
2019-06-12T19:43:20 INFO jox jox.py:274 registering the following POP of type phy as specified in jox_config: {'zone': '1', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', 'managed-by': 'none', 'pop-name': 'zone-3-phy', 'ssh-user': '', 'scope': 'private', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty', 'ssh-private-key': '', '_Comment_3': 'ssh config', 'ssh-password': '', 'domain': '172.24.10.0/24'}
2019-06-12T19:43:20 DEBUG jox jox.py:391 The phy pop with the following config is successfully regiestered: {'zone': '1', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', 'managed-by': 'none', 'pop-name': 'zone-3-phy', 'ssh-user': '', 'scope': 'private', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty', 'ssh-private-key': '', '_Comment_3': 'ssh config', 'ssh-password': '', 'domain': '172.24.10.0/24'}
2019-06-12T19:43:20 INFO jox jox.py:274 registering the following POP of type phy as specified in jox_config: {'zone': '1', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', 'managed-by': 'none', 'pop-name': 'zone-4-phy', 'ssh-user': '', 'scope': 'private', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty', 'ssh-private-key': '', '_Comment_3': 'ssh config', 'ssh-password': '', 'domain': '172.24.11.0/24'}
2019-06-12T19:43:20 DEBUG jox jox.py:391 The phy pop with the following config is successfully regiestered: {'zone': '1', '_Comment_2': 'domain is the subnet, it is optional if the endpoint and subnet-mask are provided', 'managed-by': 'none', 'pop-name': 'zone-4-phy', 'ssh-user': '', 'scope': 'private', '_Comment_1': 'pop-name is a user-defined name for the name of the pop. It will be generated randomly if it is empty', 'ssh-private-key': '', '_Comment_3': 'ssh config', 'ssh-password': '', 'domain': '172.24.11.0/24'}
2019-06-12T19:43:20 INFO jox jox.py:281 Clouds Controller is successfully loaded!
2019-06-12T19:43:20 DEBUG jox.RO.rsourceController resource_controller.py:539 jcloud localhost was added to jox
2019-06-12T19:43:20 DEBUG jox jox.py:405 The following cloud is successfully  added
2019-06-12T19:43:20 DEBUG jox.RO.rsourceController resource_controller.py:539 jcloud nymphe-edu was added to jox
2019-06-12T19:43:20 DEBUG jox jox.py:405 The following cloud is successfully  added
2019-06-12T19:43:20 INFO jox jox.py:293 All clouds were successfully added!
2019-06-12T19:43:20 INFO jox jox.py:297 Creating slice controller
2019-06-12T19:43:20 INFO jox.NetworkSliceController nsi_controller.py:41 Initial configuration of network slice controller
2019-06-12T19:43:20 INFO jox.NetworkSliceController nsi_controller.py:62 The resource controller is built
2019-06-12T19:43:20 INFO jox jox.py:301 Creating sub-slice controller
2019-06-12T19:43:20 INFO jox jox.py:309 Slices Controller was successfully loaded!
2019-06-12T19:43:20 INFO jox jox.py:314 JOX loading time: 0.30368590354919434 
2019-06-12T19:43:20 INFO jox jox.py:324 JOX Web API loaded!
2019-06-12T19:45:22 INFO jox jox.py:410 Adding the slice mosaic5g-oai-4g.yaml
2019-06-12T19:45:22 INFO jox.NetworkSliceController nsi_controller.py:102 Adding the slice mosaic5g-oai-4g.yaml
2019-06-12T19:45:22 ERROR jox.es es.py:219 Error while getting the source of the index of slice_keys_oai-4g from elasticsearch: NotFoundError(404, 'index_not_found_exception', 'no such index [slice_keys_oai-4g]')
2019-06-12T19:45:23 ERROR jox.es es.py:219 Error while getting the source of the index of slice_monitor_oai-4g from elasticsearch: NotFoundError(404, 'index_not_found_exception', 'no such index [slice_monitor_oai-4g]')
2019-06-12T19:45:23 INFO jox.templateManager template_manager.py:128 Deleting the index oai-4g from elasticsearch if alredy exist
2019-06-12T19:45:23 DEBUG jox.templateManager template_manager.py:129 Deleting the index oai-4g from elasticsearch if alredy exist
2019-06-12T19:45:23 INFO jox.templateManager template_manager.py:132 Saving the index oai-4g to elasticsearch
2019-06-12T19:45:23 DEBUG jox.templateManager template_manager.py:133 Saving the index oai-4g to elasticsearch
2019-06-12T19:45:23 ERROR jox.es es.py:219 Error while getting the source of the index of oai-4g from elasticsearch: NotFoundError(404, 'index_not_found_exception', 'no such index [oai-4g]')
2019-06-12T19:45:23 INFO jox.templateManager template_manager.py:155 Deleting the index nssi_1 from elasticsearch if already exist
2019-06-12T19:45:23 DEBUG jox.templateManager template_manager.py:156 Deleting the index nssi_1 from elasticsearch if already exist
2019-06-12T19:45:23 INFO jox.templateManager template_manager.py:161 Saving the index nssi_1 to elasticsearch
2019-06-12T19:45:23 DEBUG jox.templateManager template_manager.py:162 Saving the index nssi_1 to elasticsearch
2019-06-12T19:45:23 ERROR jox.es es.py:219 Error while getting the source of the index of nssi_1 from elasticsearch: NotFoundError(404, 'index_not_found_exception', 'no such index [nssi_1]')
2019-06-12T19:45:24 INFO jox.templateManager template_manager.py:155 Deleting the index nssi_2 from elasticsearch if already exist
2019-06-12T19:45:24 DEBUG jox.templateManager template_manager.py:156 Deleting the index nssi_2 from elasticsearch if already exist
2019-06-12T19:45:24 INFO jox.templateManager template_manager.py:161 Saving the index nssi_2 to elasticsearch
2019-06-12T19:45:24 DEBUG jox.templateManager template_manager.py:162 Saving the index nssi_2 to elasticsearch
2019-06-12T19:45:24 ERROR jox.es es.py:219 Error while getting the source of the index of nssi_2 from elasticsearch: NotFoundError(404, 'index_not_found_exception', 'no such index [nssi_2]')
2019-06-12T19:45:24 INFO jox.templateManager template_manager.py:155 Deleting the index nssi_3 from elasticsearch if already exist
2019-06-12T19:45:24 DEBUG jox.templateManager template_manager.py:156 Deleting the index nssi_3 from elasticsearch if already exist
2019-06-12T19:45:24 INFO jox.templateManager template_manager.py:161 Saving the index nssi_3 to elasticsearch
2019-06-12T19:45:24 DEBUG jox.templateManager template_manager.py:162 Saving the index nssi_3 to elasticsearch
2019-06-12T19:45:24 ERROR jox.es es.py:219 Error while getting the source of the index of nssi_3 from elasticsearch: NotFoundError(404, 'index_not_found_exception', 'no such index [nssi_3]')
2019-06-12T19:45:24 ERROR jox.es es.py:219 Error while getting the source of the index of subslice_monitor_nssi_1 from elasticsearch: NotFoundError(404, 'index_not_found_exception', 'no such index [subslice_monitor_nssi_1]')
2019-06-12T19:45:25 INFO jox.jslice slice.py:105 trying to add the subslice nssi_1
2019-06-12T19:45:25 INFO jox.subslice subslice.py:118 add the models on which the services of the  NSSI nssi_1 will be deployed
2019-06-12T19:45:26 INFO jox.jmodel model.py:143 The juju model nymphe-edu:default is configured
2019-06-12T19:45:26 DEBUG jox.jmodel model.py:144 The ID of the juju model nymphe-edu:default is f5240599-4e6f-44ff-8951-2419243bf1f6
2019-06-12T19:45:26 INFO jox.JCloud cloud.py:170 Connecting to controller
2019-06-12T19:45:26 INFO jox.JCloud cloud.py:170 Connecting to controller
2019-06-12T19:45:26 INFO jox.JCloud cloud.py:170 Connecting to controller
2019-06-12T19:45:26 INFO jox.JCloud cloud.py:170 Connecting to controller
2019-06-12T19:45:26 INFO jox.RO.rsourceController resource_controller.py:342 Adding machine for the application mysql
2019-06-12T19:45:26 INFO jox.RO.rsourceController resource_controller.py:441 Adding kvm machine whose id is: machine_name= 1:kvm, mid_ro = VDU_mysql
2019-06-12T19:45:26 DEBUG jox.KvmDriver.default vimdriver.py:723 0 time trying to connect to juju model:nymphe-edu:admin/default to add lxc machine
2019-06-12T19:45:26 DEBUG jox.KvmDriver.default vimdriver.py:726 Successful connection to the model nymphe-edu:admin/default 
2019-06-12T19:45:27 DEBUG jox.KvmDriver.default vimdriver.py:832 The machine VDU_mysql with juju id 48 is already added to juju model
2019-06-12T19:45:27 DEBUG jox.RO.rsourceController resource_controller.py:453 ('mysql', ' kvm machine added')
2019-06-12T19:45:27 INFO jox.RO.rsourceController resource_controller.py:342 Adding machine for the application oai-hss
2019-06-12T19:45:27 INFO jox.RO.rsourceController resource_controller.py:441 Adding kvm machine whose id is: machine_name= 2:kvm, mid_ro = VDU_oai_hss
2019-06-12T19:45:27 DEBUG jox.KvmDriver.default vimdriver.py:723 0 time trying to connect to juju model:nymphe-edu:admin/default to add lxc machine
2019-06-12T19:45:27 DEBUG jox.KvmDriver.default vimdriver.py:726 Successful connection to the model nymphe-edu:admin/default 
2019-06-12T19:45:27 DEBUG jox.KvmDriver.default vimdriver.py:832 The machine VDU_oai_hss with juju id 51 is already added to juju model
2019-06-12T19:45:27 DEBUG jox.RO.rsourceController resource_controller.py:453 ('oai-hss', ' kvm machine added')
2019-06-12T19:45:27 INFO jox.RO.rsourceController resource_controller.py:342 Adding machine for the application flexran
2019-06-12T19:45:27 INFO jox.RO.rsourceController resource_controller.py:441 Adding kvm machine whose id is: machine_name= 3:kvm, mid_ro = VDU_flexran
2019-06-12T19:45:27 DEBUG jox.KvmDriver.default vimdriver.py:723 0 time trying to connect to juju model:nymphe-edu:admin/default to add lxc machine
2019-06-12T19:45:27 DEBUG jox.KvmDriver.default vimdriver.py:726 Successful connection to the model nymphe-edu:admin/default 
2019-06-12T19:45:28 DEBUG jox.KvmDriver.default vimdriver.py:832 The machine VDU_flexran with juju id 49 is already added to juju model
2019-06-12T19:45:28 DEBUG jox.RO.rsourceController resource_controller.py:453 ('flexran', ' kvm machine added')
2019-06-12T19:45:28 DEBUG jox.RO.rsourceController resource_controller.py:504 Waiting for the machines to start to deploy the services
2019-06-12T19:45:28 INFO jox.jmodel model.py:202 Deploying the application mysql
2019-06-12 19:45:28,293 - jox.JujuModelServiceController - INFO - The servce mysql is deployed
2019-06-12T19:45:28 INFO jox.JujuModelServiceController juju2_plugin.py:285 The servce mysql is deployed
2019-06-12 19:45:28,293 - jox.JujuModelServiceController - DEBUG - The servce mysql is deployed
2019-06-12T19:45:28 DEBUG jox.JujuModelServiceController juju2_plugin.py:286 The servce mysql is deployed
2019-06-12T19:45:28 INFO jox.jmodel model.py:202 Deploying the application oai-hss
2019-06-12 19:45:28,533 - jox.JujuModelServiceController - INFO - The servce oai-hss is deployed
2019-06-12T19:45:28 INFO jox.JujuModelServiceController juju2_plugin.py:285 The servce oai-hss is deployed
2019-06-12 19:45:28,533 - jox.JujuModelServiceController - DEBUG - The servce oai-hss is deployed
2019-06-12T19:45:28 DEBUG jox.JujuModelServiceController juju2_plugin.py:286 The servce oai-hss is deployed
2019-06-12T19:45:28 INFO jox.jmodel model.py:202 Deploying the application flexran
2019-06-12 19:45:28,772 - jox.JujuModelServiceController - INFO - Response from plgin -> {"status_code": "OK", "data": "FlexRAN endpoint updated", "ACK": true}
2019-06-12T19:45:28 INFO jox.JujuModelServiceController juju2_plugin.py:369 Response from plgin -> {"status_code": "OK", "data": "FlexRAN endpoint updated", "ACK": true}
2019-06-12 19:45:28,772 - jox.JujuModelServiceController - INFO - The servce flexran is deployed
2019-06-12T19:45:28 INFO jox.JujuModelServiceController juju2_plugin.py:285 The servce flexran is deployed
2019-06-12 19:45:28,772 - jox.JujuModelServiceController - DEBUG - The servce flexran is deployed
2019-06-12T19:45:28 DEBUG jox.JujuModelServiceController juju2_plugin.py:286 The servce flexran is deployed
2019-06-12T19:45:28 INFO jox.jmodel model.py:237 Adding relation between mysql and oai-hss
2019-06-12T19:45:29 DEBUG jox.subslice subslice.py:236 The model default under the juju controller nymphe-edu is already added
2019-06-12T19:45:29 DEBUG jox.subslice subslice.py:236 The model default under the juju controller nymphe-edu is already added
2019-06-12T19:45:29 INFO jox.nssiController nssi_controller.py:129 Slice nssi_1 was added
2019-06-12T19:45:29 INFO jox.jslice slice.py:108 the subslice nssi_1 is successfully added
2019-06-12T19:45:29 ERROR jox.es es.py:219 Error while getting the source of the index of subslice_monitor_nssi_2 from elasticsearch: NotFoundError(404, 'index_not_found_exception', 'no such index [subslice_monitor_nssi_2]')
2019-06-12T19:45:29 INFO jox.jslice slice.py:105 trying to add the subslice nssi_2
2019-06-12T19:45:29 INFO jox.subslice subslice.py:118 add the models on which the services of the  NSSI nssi_2 will be deployed
2019-06-12T19:45:30 INFO jox.jmodel model.py:143 The juju model nymphe-edu:default is configured
2019-06-12T19:45:30 DEBUG jox.jmodel model.py:144 The ID of the juju model nymphe-edu:default is f5240599-4e6f-44ff-8951-2419243bf1f6
2019-06-12T19:45:30 INFO jox.RO.rsourceController resource_controller.py:342 Adding machine for the application oai-spgw
2019-06-12T19:45:30 INFO jox.RO.rsourceController resource_controller.py:441 Adding kvm machine whose id is: machine_name= 4:kvm, mid_ro = VDU_oai-spgw
2019-06-12T19:45:30 DEBUG jox.KvmDriver.default vimdriver.py:723 0 time trying to connect to juju model:nymphe-edu:admin/default to add lxc machine
2019-06-12T19:45:30 DEBUG jox.KvmDriver.default vimdriver.py:726 Successful connection to the model nymphe-edu:admin/default 
2019-06-12T19:45:30 DEBUG jox.KvmDriver.default vimdriver.py:832 The machine VDU_oai-spgw with juju id 50 is already added to juju model
2019-06-12T19:45:30 DEBUG jox.RO.rsourceController resource_controller.py:453 ('oai-spgw', ' kvm machine added')
2019-06-12T19:45:30 INFO jox.RO.rsourceController resource_controller.py:342 Adding machine for the application oai-mme
2019-06-12T19:45:30 INFO jox.RO.rsourceController resource_controller.py:441 Adding kvm machine whose id is: machine_name= 5:kvm, mid_ro = VDU_oai-mme
2019-06-12T19:45:30 DEBUG jox.KvmDriver.default vimdriver.py:723 0 time trying to connect to juju model:nymphe-edu:admin/default to add lxc machine
2019-06-12T19:45:30 DEBUG jox.KvmDriver.default vimdriver.py:726 Successful connection to the model nymphe-edu:admin/default 
2019-06-12T19:45:31 DEBUG jox.KvmDriver.default vimdriver.py:832 The machine VDU_oai-mme with juju id 47 is already added to juju model
2019-06-12T19:45:31 DEBUG jox.RO.rsourceController resource_controller.py:453 ('oai-mme', ' kvm machine added')
2019-06-12T19:45:31 DEBUG jox.RO.rsourceController resource_controller.py:504 Waiting for the machines to start to deploy the services
2019-06-12T19:45:31 INFO jox.jmodel model.py:202 Deploying the application oai-spgw
2019-06-12 19:45:31,365 - jox.JujuModelServiceController - INFO - The servce oai-spgw is deployed
2019-06-12 19:45:31,365 - jox.JujuModelServiceController - INFO - The servce oai-spgw is deployed
2019-06-12T19:45:31 INFO jox.JujuModelServiceController juju2_plugin.py:285 The servce oai-spgw is deployed
2019-06-12 19:45:31,366 - jox.JujuModelServiceController - DEBUG - The servce oai-spgw is deployed
2019-06-12 19:45:31,366 - jox.JujuModelServiceController - DEBUG - The servce oai-spgw is deployed
2019-06-12T19:45:31 DEBUG jox.JujuModelServiceController juju2_plugin.py:286 The servce oai-spgw is deployed
2019-06-12T19:45:31 INFO jox.jmodel model.py:202 Deploying the application oai-mme
2019-06-12 19:45:31,607 - jox.JujuModelServiceController - INFO - The servce oai-mme is deployed
2019-06-12 19:45:31,607 - jox.JujuModelServiceController - INFO - The servce oai-mme is deployed
2019-06-12T19:45:31 INFO jox.JujuModelServiceController juju2_plugin.py:285 The servce oai-mme is deployed
2019-06-12 19:45:31,607 - jox.JujuModelServiceController - DEBUG - The servce oai-mme is deployed
2019-06-12 19:45:31,607 - jox.JujuModelServiceController - DEBUG - The servce oai-mme is deployed
2019-06-12T19:45:31 DEBUG jox.JujuModelServiceController juju2_plugin.py:286 The servce oai-mme is deployed
2019-06-12T19:45:31 INFO jox.jmodel model.py:237 Adding relation between oai-mme and oai-spgw
2019-06-12T19:45:31 DEBUG jox.subslice subslice.py:236 The model default under the juju controller nymphe-edu is already added
2019-06-12T19:45:31 INFO jox.nssiController nssi_controller.py:129 Slice nssi_2 was added
2019-06-12T19:45:31 INFO jox.jslice slice.py:108 the subslice nssi_2 is successfully added
2019-06-12T19:45:31 ERROR jox.es es.py:219 Error while getting the source of the index of subslice_monitor_nssi_3 from elasticsearch: NotFoundError(404, 'index_not_found_exception', 'no such index [subslice_monitor_nssi_3]')
2019-06-12T19:45:32 INFO jox.jslice slice.py:105 trying to add the subslice nssi_3
2019-06-12T19:45:32 INFO jox.subslice subslice.py:118 add the models on which the services of the  NSSI nssi_3 will be deployed
2019-06-12T19:45:33 INFO jox.jmodel model.py:143 The juju model nymphe-edu:default is configured
2019-06-12T19:45:33 DEBUG jox.jmodel model.py:144 The ID of the juju model nymphe-edu:default is f5240599-4e6f-44ff-8951-2419243bf1f6
2019-06-12T19:45:33 INFO jox.RO.rsourceController resource_controller.py:342 Adding machine for the application oai-ran
2019-06-12T19:45:33 INFO jox.RO.rsourceController resource_controller.py:486 Adding physical machine whose id is: machine_name= 6:phy, mid_ro = VDU_oai-ran
2019-06-12T19:45:33 DEBUG jox.PhyDriver.zone-1-phy vimdriver.py:1079 0 time trying to connect to juju model:nymphe-edu:admin/default to add lxc machine
2019-06-12T19:45:33 DEBUG jox.PhyDriver.zone-1-phy vimdriver.py:1082 Successful connection to the model nymphe-edu:admin/default 
2019-06-12T19:45:33 DEBUG jox.RO.rsourceController resource_controller.py:498 ('oai-ran', ' kvm machine added')
2019-06-12T19:45:33 DEBUG jox.RO.rsourceController resource_controller.py:504 Waiting for the machines to start to deploy the services
2019-06-12T19:45:33 INFO jox.jmodel model.py:202 Deploying the application oai-ran
2019-06-12 19:45:42,357 - jox.JujuModelServiceController - INFO - The servce oai-ran is deployed
2019-06-12 19:45:42,357 - jox.JujuModelServiceController - INFO - The servce oai-ran is deployed
2019-06-12 19:45:42,357 - jox.JujuModelServiceController - INFO - The servce oai-ran is deployed
2019-06-12T19:45:42 INFO jox.JujuModelServiceController juju2_plugin.py:285 The servce oai-ran is deployed
2019-06-12 19:45:42,359 - jox.JujuModelServiceController - DEBUG - The servce oai-ran is deployed
2019-06-12 19:45:42,359 - jox.JujuModelServiceController - DEBUG - The servce oai-ran is deployed
2019-06-12 19:45:42,359 - jox.JujuModelServiceController - DEBUG - The servce oai-ran is deployed
2019-06-12T19:45:42 DEBUG jox.JujuModelServiceController juju2_plugin.py:286 The servce oai-ran is deployed
2019-06-12T19:45:42 INFO jox.nssiController nssi_controller.py:129 Slice nssi_3 was added
2019-06-12T19:45:42 INFO jox.jslice slice.py:108 the subslice nssi_3 is successfully added
2019-06-12T19:45:42 INFO jox.jslice slice.py:115 adding entre-nssi relations
2019-06-12T19:45:42 INFO jox.jslice slice.py:81 getting the inter relations of the slice oai-4g
2019-06-12T19:45:42 DEBUG jox.jslice slice.py:156 adding relation between oai-mme from the nssi nssi_2 and oai-ran from the nssi nssi_3
2019-06-12T19:45:52 DEBUG jox.jslice slice.py:156 adding relation between oai-ran from the nssi nssi_3 and flexran from the nssi nssi_1
2019-06-12T19:46:03 DEBUG jox.jslice slice.py:156 adding relation between oai-hss from the nssi nssi_1 and oai-mme from the nssi nssi_2
2019-06-12T19:46:11 ERROR jox.monitor monitor.py:354 The key service_keys does not exist in the page slice_keys_oai-4g
2019-06-12T19:46:13 ERROR jox.monitor monitor.py:354 The key service_keys does not exist in the page slice_keys_oai-4g
2019-06-12T19:46:13 ERROR jox.monitor monitor.py:316 The key service_keys does not exist in the page slice_keys_oai-4g
2019-06-12T19:46:13 INFO jox.NetworkSliceController nsi_controller.py:109 The slice mosaic5g-oai-4g.yaml is successfully added
2019-06-12T19:46:19 ERROR jox.monitor monitor.py:354 The key service_keys does not exist in the page slice_keys_oai-4g
2019-06-12T19:46:57 ERROR jox.monitor monitor.py:354 The key service_keys does not exist in the page slice_keys_oai-4g
2019-06-12T19:46:57 ERROR jox.monitor monitor.py:316 The key service_keys does not exist in the page slice_keys_oai-4g
2019-06-12T19:47:05 ERROR jox.monitor monitor.py:354 The key service_keys does not exist in the page slice_keys_oai-4g
2019-06-12T19:47:05 ERROR jox.monitor monitor.py:354 The key service_keys does not exist in the page slice_keys_oai-4g
2019-06-12T19:47:08 ERROR jox.monitor monitor.py:354 The key service_keys does not exist in the page slice_keys_oai-4g
2019-06-12T19:47:08 ERROR jox.monitor monitor.py:354 The key service_keys does not exist in the page slice_keys_oai-4g
